.github/workflows/ci.yml

```diff
@@ -187,7 +187,7 @@ jobs:
           sudo apt-get install -y nodejs
           sudo npm install -g yarn
 
-          pushd integration-tests/
+          pushd java-script/
 
           # prepare typescript environment and wait for creditcoin-node to initialize
           ./prepare-devel-env.sh
```

Comment:
After the merge of the 2 directories I've kept `src/interfaces/` from the original `polkadotjs-examples/` directory b/c that was stored in git. My original understanding is that @jfrank-summit wanted to make it easier for newcomers to get started. 

The script above explicitly generates type definitions from the blockchain metadata b/c in the original `integration-tests/` directory these files were `.gitignore`-d. Currently this results in the following difference: 

```bash
$ git status
On branch tests/unify-tests-and-examples
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   creditcoin.json
	modified:   src/interfaces/augment-api-errors.ts
	modified:   src/interfaces/augment-api-events.ts
	modified:   src/interfaces/augment-api-rpc.ts
	modified:   src/interfaces/augment-types.ts
	modified:   src/interfaces/lookup.ts
	modified:   src/interfaces/types-lookup.ts
```
and

```diff
diff --git a/java-script/creditcoin.json b/java-script/creditcoin.json
index 5cf168b..e899d1b 100644
--- a/java-script/creditcoin.json
+++ b/java-script/creditcoin.json
@@ -1,5 +1 @@
-{
-    "jsonrpc": "2.0",
-    "result": "0x6d6574610e3902000c1c73705f636f72651863727970746f2c4163636f756e7449643332000004000401205b75383b2033325d0000040000032000000008000800000503000c08306672616d655f73797374656d2c4163636f756e74496e666f0814496e64657801102c4163636f756e74446174610114001401146e6f6e6365100114496e646578000124636f6e73756d657273100120526566436f756e7400012470726f766964657273100120526566436f756e7400012c73756666696369656e7473100120526566436f756e740001106461746114012c4163636f756e7444617461000010000005050014083c70616c6c65745f62616c616e6365732c4163636f756e7444617461041c42616c616e63650118001001106672656518011c42616c616e6365000120726573657276656418011c42616c616e636500012c6d6973635f66726f7a656e18011c42616c616e63650001286665655f66726f7a656e18011c42616c616e636500001800000507001c0c346672616d655f737570706f72741c77656967687473405065724469737061746368436c6173730404540120000c01186e6f726d616c2001045400012c6f7065726174696f6e616c200104540001246d616e6461746f727920010454000020000005060024083c7072696d69746976655f74797065731048323536000004000401205b75383b2033325d00002800000208002c102873705f72756e74696d651c67656e65726963186469676573741844696765737400000401106c6f677330013c5665633c4469676573744974656d3e000030000002340034102873705f72756e74696d651c67656e6572696318646967657374284469676573744974656d0001142850726552756e74696d650800380144436f6e73656e737573456e67696e654964000028011c5665633c75383e00060024436f6e73656e7375730800380144436f6e73656e737573456e67696e654964000028011c5665633c75383e000400105365616c0800380144436f6e73656e737573456e67696e654964000028011c5665633c75383e000500144f74686572040028011c5665633c75383e0000006452756e74696d65456e7669726f6e6d656e745570646174656400080000380000030400000008003c00000240004008306672616d655f73797374656d2c4576656e745265636f7264080445014404540124000c01147068617365ec011450686173650001146576656e7444010445000118746f70696373f001185665633c543e000044085c637265646974636f696e5f6e6f64655f72756e74696d65144576656e740001141853797374656d04004801706672616d655f73797374656d3a3a4576656e743c52756e74696d653e0000002042616c616e636573040064017c70616c6c65745f62616c616e6365733a3a4576656e743c52756e74696d653e000200105375646f04006c016c70616c6c65745f7375646f3a3a4576656e743c52756e74696d653e00040028437265646974636f696e04007c018470616c6c65745f637265646974636f696e3a3a4576656e743c52756e74696d653e0005001c526577617264730400e8017870616c6c65745f726577617264733a3a4576656e743c52756e74696d653e00070000480c306672616d655f73797374656d1870616c6c6574144576656e740404540001184045787472696e7369635375636365737304013464697370617463685f696e666f4c01304469737061746368496e666f00000490416e2065787472696e73696320636f6d706c65746564207375636365737366756c6c792e3c45787472696e7369634661696c656408013864697370617463685f6572726f7258013444697370617463684572726f7200013464697370617463685f696e666f4c01304469737061746368496e666f00010450416e2065787472696e736963206661696c65642e2c436f64655570646174656400020450603a636f6465602077617320757064617465642e284e65774163636f756e7404011c6163636f756e74000130543a3a4163636f756e7449640003046841206e6577206163636f756e742077617320637265617465642e344b696c6c65644163636f756e7404011c6163636f756e74000130543a3a4163636f756e74496400040458416e206163636f756e7420776173207265617065642e2052656d61726b656408011873656e646572000130543a3a4163636f756e7449640001106861736824011c543a3a48617368000504704f6e206f6e2d636861696e2072656d61726b2068617070656e65642e04704576656e7420666f72207468652053797374656d2070616c6c65742e4c0c346672616d655f737570706f72741c77656967687473304469737061746368496e666f00000c0118776569676874200118576569676874000114636c6173735001344469737061746368436c617373000120706179735f666565540110506179730000500c346672616d655f737570706f72741c77656967687473344469737061746368436c61737300010c184e6f726d616c0000002c4f7065726174696f6e616c000100244d616e6461746f727900020000540c346672616d655f737570706f72741c7765696768747310506179730001080c596573000000084e6f0001000058082873705f72756e74696d653444697370617463684572726f72000124144f746865720000003043616e6e6f744c6f6f6b7570000100244261644f726967696e000200184d6f64756c65080114696e64657808010875380001146572726f72080108753800030044436f6e73756d657252656d61696e696e670004002c4e6f50726f76696465727300050040546f6f4d616e79436f6e73756d65727300060014546f6b656e04005c0128546f6b656e4572726f720007002841726974686d65746963040060013c41726974686d657469634572726f72000800005c082873705f72756e74696d6528546f6b656e4572726f7200011c1c4e6f46756e647300000020576f756c644469650001003042656c6f774d696e696d756d0002003043616e6e6f7443726561746500030030556e6b6e6f776e41737365740004001846726f7a656e0005002c556e737570706f727465640006000060082873705f72756e74696d653c41726974686d657469634572726f7200010c24556e646572666c6f77000000204f766572666c6f77000100384469766973696f6e42795a65726f00020000640c3c70616c6c65745f62616c616e6365731870616c6c6574144576656e740804540004490001281c456e646f77656408011c6163636f756e74000130543a3a4163636f756e744964000130667265655f62616c616e6365180128543a3a42616c616e6365000004b8416e206163636f756e74207761732063726561746564207769746820736f6d6520667265652062616c616e63652e20447573744c6f737408011c6163636f756e74000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e63650001083d01416e206163636f756e74207761732072656d6f7665642077686f73652062616c616e636520776173206e6f6e2d7a65726f206275742062656c6f77204578697374656e7469616c4465706f7369742c78726573756c74696e6720696e20616e206f75747269676874206c6f73732e205472616e736665720c011066726f6d000130543a3a4163636f756e744964000108746f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e63650002044c5472616e73666572207375636365656465642e2842616c616e63655365740c010c77686f000130543a3a4163636f756e74496400011066726565180128543a3a42616c616e63650001207265736572766564180128543a3a42616c616e636500030468412062616c616e6365207761732073657420627920726f6f742e20526573657276656408010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e6365000404e0536f6d652062616c616e63652077617320726573657276656420286d6f7665642066726f6d206672656520746f207265736572766564292e28556e726573657276656408010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e6365000504e8536f6d652062616c616e63652077617320756e726573657276656420286d6f7665642066726f6d20726573657276656420746f2066726565292e4852657365727665526570617472696174656410011066726f6d000130543a3a4163636f756e744964000108746f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e636500014864657374696e6174696f6e5f7374617475736801185374617475730006084d01536f6d652062616c616e636520776173206d6f7665642066726f6d207468652072657365727665206f6620746865206669727374206163636f756e7420746f20746865207365636f6e64206163636f756e742ed846696e616c20617267756d656e7420696e64696361746573207468652064657374696e6174696f6e2062616c616e636520747970652e1c4465706f73697408010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e6365000704d8536f6d6520616d6f756e7420776173206465706f73697465642028652e672e20666f72207472616e73616374696f6e2066656573292e20576974686472617708010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e63650008041d01536f6d6520616d6f756e74207761732077697468647261776e2066726f6d20746865206163636f756e742028652e672e20666f72207472616e73616374696f6e2066656573292e1c536c617368656408010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e63650009040101536f6d6520616d6f756e74207761732072656d6f7665642066726f6d20746865206163636f756e742028652e672e20666f72206d69736265686176696f72292e0499010a090909546865205b6576656e745d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f72732920656d69747465640a090909627920746869732070616c6c65742e0a0909096814346672616d655f737570706f72741874726169747318746f6b656e73106d6973633442616c616e63655374617475730001081046726565000000205265736572766564000100006c0c2c70616c6c65745f7375646f1870616c6c6574144576656e7404045400010c14537564696404012c7375646f5f726573756c747001384469737061746368526573756c740000048841207375646f206a75737420746f6f6b20706c6163652e205c5b726573756c745c5d284b65794368616e6765640401286f6c645f7375646f65727801504f7074696f6e3c543a3a4163636f756e7449643e0001043901546865205c5b7375646f65725c5d206a757374207377697463686564206964656e746974793b20746865206f6c64206b657920697320737570706c696564206966206f6e6520657869737465642e285375646f4173446f6e6504012c7375646f5f726573756c747001384469737061746368526573756c740002048841207375646f206a75737420746f6f6b20706c6163652e205c5b726573756c745c5d0499010a090909546865205b6576656e745d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f72732920656d69747465640a090909627920746869732070616c6c65742e0a090909700418526573756c740804540174044501580108084f6b040074000000000c45727204005800000100007400000400007804184f7074696f6e04045401000108104e6f6e6500000010536f6d6504000000000100007c0c4470616c6c65745f637265646974636f696e1870616c6c6574144576656e7404045400013044416464726573735265676973746572656408008001484164647265737349643c543a3a486173683e0000840154416464726573733c543a3a4163636f756e7449643e00000859014576656e7420646f63756d656e746174696f6e2073686f756c6420656e64207769746820616e20617272617920746861742070726f7669646573206465736372697074697665206e616d657320666f72206576656e7470706172616d65746572732e205b736f6d657468696e672c2077686f5d485472616e7366657252656769737465726564080090014c5472616e7366657249643c543a3a486173683e00009401bc5472616e736665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e000100405472616e736665725665726966696564080090014c5472616e7366657249643c543a3a486173683e00009401bc5472616e736665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e000200445472616e7366657250726f636573736564080090014c5472616e7366657249643c543a3a486173683e00009401bc5472616e736665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e0003003441736b4f7264657241646465640800b4018c41736b4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000b801e841736b4f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e000400344269644f7264657241646465640800c4018c4269644f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000c801e84269644f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e000500284f6666657241646465640800d001804f6666657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000d401b04f666665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e000600384465616c4f7264657241646465640800a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000d801ec4465616c4f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e0007003c4465616c4f7264657246756e6465640800a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000d801ec4465616c4f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e0008003c4465616c4f72646572436c6f7365640800a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000d801ec4465616c4f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e000900304c6f616e4578656d707465640800a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e000090014c5472616e7366657249643c543a3a486173683e000a004c4c656761637957616c6c6574436c61696d65640c00000130543a3a4163636f756e7449640000e001344c6567616379536967686173680000180128543a3a42616c616e6365000b000499010a090909546865205b6576656e745d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f72732920656d69747465640a090909627920746869732070616c6c65742e0a090909800c4470616c6c65745f637265646974636f696e147479706573244164647265737349640410486173680124000400240110486173680000840c4470616c6c65745f637265646974636f696e1474797065731c4164647265737304244163636f756e7449640100000c0128626c6f636b636861696e880128426c6f636b636861696e00011476616c75658c013c45787465726e616c416464726573730001146f776e65720001244163636f756e7449640000880c4470616c6c65745f637265646974636f696e14747970657328426c6f636b636861696e00011420457468657265756d0000001c52696e6b656279000100244c756e6976657273650002001c426974636f696e000300144f7468657204008c01284f74686572436861696e000400008c10346672616d655f737570706f72741c73746f726167652c626f756e6465645f76656328426f756e64656456656308045401080453000004002801185665633c543e0000900c4470616c6c65745f637265646974636f696e147479706573285472616e7366657249640410486173680124000400240110486173680000940c4470616c6c65745f637265646974636f696e147479706573205472616e736665720c244163636f756e744964010020426c6f636b4e756d01101048617368012400280128626c6f636b636861696e880128426c6f636b636861696e0001106b696e649801305472616e736665724b696e6400011066726f6d80013c4164647265737349643c486173683e000108746f80013c4164647265737349643c486173683e0001206f726465725f69649c015c4f7264657249643c426c6f636b4e756d2c20486173683e000118616d6f756e74a8013845787465726e616c416d6f756e7400010874788c013045787465726e616c54784964000114626c6f636b100120426c6f636b4e756d00012470726f636573736564b00110626f6f6c00011c736967686173680001244163636f756e7449640000980c4470616c6c65745f637265646974636f696e147479706573305472616e736665724b696e6400011014457263323004008c013c45787465726e616c416464726573730000001c4574686c65737304008c013c45787465726e616c41646472657373000100184e6174697665000200144f7468657204008c01444f746865725472616e736665724b696e64000300009c0c4470616c6c65745f637265646974636f696e1474797065731c4f7264657249640820426c6f636b4e756d0110104861736801240108104465616c0400a0016c4465616c4f7264657249643c426c6f636b4e756d2c20486173683e0000002452657061796d656e740400a4018052657061796d656e744f7264657249643c426c6f636b4e756d2c20486173683e00010000a00c4470616c6c65745f637265646974636f696e1474797065732c4465616c4f7264657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000a40c4470616c6c65745f637265646974636f696e1474797065734052657061796d656e744f7264657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000a8083c7072696d69746976655f7479706573105532353600000400ac01205b7536343b20345d0000ac000003040000002000b00000050000b40c4470616c6c65745f637265646974636f696e1474797065732841736b4f7264657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000b80c4470616c6c65745f637265646974636f696e1474797065732041736b4f7264657210244163636f756e744964010020426c6f636b4e756d011010486173680124184d6f6d656e74012000180128626c6f636b636861696e880128426c6f636b636861696e0001446c656e6465725f616464726573735f696480013c4164647265737349643c486173683e0001147465726d73bc014041736b5465726d733c4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100120426c6f636b4e756d000114626c6f636b100120426c6f636b4e756d0001186c656e6465720001244163636f756e7449640000bc104470616c6c65745f637265646974636f696e147479706573286c6f616e5f7465726d732041736b5465726d7304184d6f6d656e740120000400c001444c6f616e5465726d733c4d6f6d656e743e0000c0104470616c6c65745f637265646974636f696e147479706573286c6f616e5f7465726d73244c6f616e5465726d7304184d6f6d656e740120000c0118616d6f756e74a8013845787465726e616c416d6f756e74000134696e7465726573745f72617465200130496e746572657374526174650001206d617475726974792001184d6f6d656e740000c40c4470616c6c65745f637265646974636f696e147479706573284269644f7264657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000c80c4470616c6c65745f637265646974636f696e147479706573204269644f7264657210244163636f756e744964010020426c6f636b4e756d011010486173680124184d6f6d656e74012000180128626c6f636b636861696e880128426c6f636b636861696e00014c626f72726f7765725f616464726573735f696480013c4164647265737349643c486173683e0001147465726d73cc01404269645465726d733c4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100120426c6f636b4e756d000114626c6f636b100120426c6f636b4e756d000120626f72726f7765720001244163636f756e7449640000cc104470616c6c65745f637265646974636f696e147479706573286c6f616e5f7465726d73204269645465726d7304184d6f6d656e740120000400c001444c6f616e5465726d733c4d6f6d656e743e0000d00c4470616c6c65745f637265646974636f696e1474797065731c4f6666657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000d40c4470616c6c65745f637265646974636f696e147479706573144f666665720c244163636f756e744964010020426c6f636b4e756d01101048617368012400180128626c6f636b636861696e880128426c6f636b636861696e00011861736b5f6964b4016841736b4f7264657249643c426c6f636b4e756d2c20486173683e0001186269645f6964c401684269644f7264657249643c426c6f636b4e756d2c20486173683e00014065787069726174696f6e5f626c6f636b100120426c6f636b4e756d000114626c6f636b100120426c6f636b4e756d0001186c656e6465720001244163636f756e7449640000d80c4470616c6c65745f637265646974636f696e147479706573244465616c4f7264657210244163636f756e744964010020426c6f636b4e756d011010486173680124184d6f6d656e740120002c0128626c6f636b636861696e880128426c6f636b636861696e0001206f666665725f6964d0015c4f6666657249643c426c6f636b4e756d2c20486173683e0001446c656e6465725f616464726573735f696480013c4164647265737349643c486173683e00014c626f72726f7765725f616464726573735f696480013c4164647265737349643c486173683e0001147465726d73c001444c6f616e5465726d733c4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100120426c6f636b4e756d00012474696d657374616d702001184d6f6d656e7400014c66756e64696e675f7472616e736665725f6964dc01604f7074696f6e3c5472616e7366657249643c486173683e3e00015472657061796d656e745f7472616e736665725f6964dc01604f7074696f6e3c5472616e7366657249643c486173683e3e0001106c6f636b7801444f7074696f6e3c4163636f756e7449643e000120626f72726f7765720001244163636f756e7449640000dc04184f7074696f6e04045401900108104e6f6e6500000010536f6d650400900000010000e00c4470616c6c65745f637265646974636f696e147479706573344c65676163795369676861736800000400e401205b75383b2036305d0000e40000033c0000000800e80c3870616c6c65745f726577617264731870616c6c6574144576656e740404540001043052657761726449737375656408000001384163636f756e7449644f663c543e000018013042616c616e63654f663c543e000004a452657761726420776173206973737565642e205b626c6f636b5f617574686f722c20616d6f756e745d0499010a090909546865205b6576656e745d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f72732920656d69747465640a090909627920746869732070616c6c65742e0a090909ec08306672616d655f73797374656d14506861736500010c384170706c7945787472696e736963040010010c7533320000003046696e616c697a6174696f6e00010038496e697469616c697a6174696f6e00020000f00000022400f4000002f800f800000408101000fc08306672616d655f73797374656d584c61737452756e74696d6555706772616465496e666f0000080130737065635f76657273696f6e0101014c636f6465633a3a436f6d706163743c7533323e000124737065635f6e616d650501016473705f72756e74696d653a3a52756e74696d65537472696e670000010100000610000501000005020009010c306672616d655f73797374656d1870616c6c65741043616c6c0404540001242866696c6c5f626c6f636b040114726174696f0d01011c50657262696c6c00000405014120646973706174636820746861742077696c6c2066696c6c2074686520626c6f636b2077656967687420757020746f2074686520676976656e20726174696f2e1872656d61726b04011872656d61726b28011c5665633c75383e000114684d616b6520736f6d65206f6e2d636861696e2072656d61726b2e002823203c7765696768743e202d20604f283129602c23203c2f7765696768743e387365745f686561705f7061676573040114706167657320010c753634000204f853657420746865206e756d626572206f6620706167657320696e2074686520576562417373656d626c7920656e7669726f6e6d656e74277320686561702e207365745f636f6465040110636f646528011c5665633c75383e0003306453657420746865206e65772072756e74696d6520636f64652e002823203c7765696768743e31012d20604f2843202b2053296020776865726520604360206c656e677468206f662060636f64656020616e642060536020636f6d706c6578697479206f66206063616e5f7365745f636f64656045012d20312063616c6c20746f206063616e5f7365745f636f6465603a20604f28532960202863616c6c73206073705f696f3a3a6d6973633a3a72756e74696d655f76657273696f6e60207768696368206973342020657870656e73697665292e842d20312073746f726167652077726974652028636f64656320604f28432960292e402d203120646967657374206974656d2e282d2031206576656e742e4d0154686520776569676874206f6620746869732066756e6374696f6e20697320646570656e64656e74206f6e207468652072756e74696d652c206275742067656e6572616c6c7920746869732069732076657279b8657870656e736976652e2057652077696c6c207472656174207468697320617320612066756c6c20626c6f636b2e2c23203c2f7765696768743e5c7365745f636f64655f776974686f75745f636865636b73040110636f646528011c5665633c75383e000424190153657420746865206e65772072756e74696d6520636f646520776974686f757420646f696e6720616e7920636865636b73206f662074686520676976656e2060636f6465602e002823203c7765696768743e8c2d20604f2843296020776865726520604360206c656e677468206f662060636f646560842d20312073746f726167652077726974652028636f64656320604f28432960292e402d203120646967657374206974656d2e282d2031206576656e742e550154686520776569676874206f6620746869732066756e6374696f6e20697320646570656e64656e74206f6e207468652072756e74696d652e2057652077696c6c207472656174207468697320617320612066756c6c48626c6f636b2e2023203c2f7765696768743e2c7365745f73746f726167650401146974656d73110101345665633c4b657956616c75653e0005046853657420736f6d65206974656d73206f662073746f726167652e306b696c6c5f73746f726167650401106b657973190101205665633c4b65793e000604744b696c6c20736f6d65206974656d732066726f6d2073746f726167652e2c6b696c6c5f70726566697808011870726566697828010c4b657900011c7375626b65797310010c75333200071011014b696c6c20616c6c2073746f72616765206974656d7320776974682061206b657920746861742073746172747320776974682074686520676976656e207072656669782e0039012a2a4e4f54453a2a2a2057652072656c79206f6e2074686520526f6f74206f726967696e20746f2070726f7669646520757320746865206e756d626572206f66207375626b65797320756e6465723d0174686520707265666978207765206172652072656d6f76696e6720746f2061636375726174656c792063616c63756c6174652074686520776569676874206f6620746869732066756e6374696f6e2e4472656d61726b5f776974685f6576656e7404011872656d61726b28011c5665633c75383e000818a44d616b6520736f6d65206f6e2d636861696e2072656d61726b20616e6420656d6974206576656e742e002823203c7765696768743eb42d20604f28622960207768657265206220697320746865206c656e677468206f66207468652072656d61726b2e282d2031206576656e742e2c23203c2f7765696768743e042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632e0d010c3473705f61726974686d65746963287065725f7468696e67731c50657262696c6c0000040010010c75333200001101000002150100150100000408282800190100000228001d010c306672616d655f73797374656d186c696d69747330426c6f636b5765696768747300000c0128626173655f626c6f636b2001185765696768740001246d61785f626c6f636b2001185765696768740001247065725f636c617373210101845065724469737061746368436c6173733c57656967687473506572436c6173733e000021010c346672616d655f737570706f72741c77656967687473405065724469737061746368436c617373040454012501000c01186e6f726d616c250101045400012c6f7065726174696f6e616c25010104540001246d616e6461746f72792501010454000025010c306672616d655f73797374656d186c696d6974733c57656967687473506572436c6173730000100138626173655f65787472696e7369632001185765696768740001346d61785f65787472696e736963290101384f7074696f6e3c5765696768743e0001246d61785f746f74616c290101384f7074696f6e3c5765696768743e0001207265736572766564290101384f7074696f6e3c5765696768743e0000290104184f7074696f6e04045401200108104e6f6e6500000010536f6d6504002000000100002d010c306672616d655f73797374656d186c696d6974732c426c6f636b4c656e677468000004010c6d6178310101545065724469737061746368436c6173733c7533323e000031010c346672616d655f737570706f72741c77656967687473405065724469737061746368436c6173730404540110000c01186e6f726d616c1001045400012c6f7065726174696f6e616c100104540001246d616e6461746f727910010454000035010c346672616d655f737570706f72741c776569676874733c52756e74696d654462576569676874000008011072656164200118576569676874000114777269746520011857656967687400003901082873705f76657273696f6e3852756e74696d6556657273696f6e0000200124737065635f6e616d650501013452756e74696d65537472696e67000124696d706c5f6e616d650501013452756e74696d65537472696e67000144617574686f72696e675f76657273696f6e10010c753332000130737065635f76657273696f6e10010c753332000130696d706c5f76657273696f6e10010c753332000110617069733d01011c4170697356656300014c7472616e73616374696f6e5f76657273696f6e10010c75333200013473746174655f76657273696f6e080108753800003d01040c436f77040454014101000400410100000041010000024501004501000004084901100049010000030800000008004d01000005040051010c306672616d655f73797374656d1870616c6c6574144572726f720404540001183c496e76616c6964537065634e616d650000081101546865206e616d65206f662073706563696669636174696f6e20646f6573206e6f74206d61746368206265747765656e207468652063757272656e742072756e74696d6550616e6420746865206e65772072756e74696d652e685370656356657273696f6e4e65656473546f496e63726561736500010841015468652073706563696669636174696f6e2076657273696f6e206973206e6f7420616c6c6f77656420746f206465637265617365206265747765656e207468652063757272656e742072756e74696d6550616e6420746865206e65772072756e74696d652e744661696c6564546f4578747261637452756e74696d6556657273696f6e00020cec4661696c656420746f2065787472616374207468652072756e74696d652076657273696f6e2066726f6d20746865206e65772072756e74696d652e0009014569746865722063616c6c696e672060436f72655f76657273696f6e60206f72206465636f64696e67206052756e74696d6556657273696f6e60206661696c65642e4c4e6f6e44656661756c74436f6d706f73697465000304fc537569636964652063616c6c6564207768656e20746865206163636f756e7420686173206e6f6e2d64656661756c7420636f6d706f7369746520646174612e3c4e6f6e5a65726f526566436f756e74000404350154686572652069732061206e6f6e2d7a65726f207265666572656e636520636f756e742070726576656e74696e6720746865206163636f756e742066726f6d206265696e67207075726765642e3043616c6c46696c7465726564000504d0546865206f726967696e2066696c7465722070726576656e74207468652063616c6c20746f20626520646973706174636865642e046c4572726f7220666f72207468652053797374656d2070616c6c657455010c4070616c6c65745f74696d657374616d701870616c6c65741043616c6c0404540001040c73657404010c6e6f7759010124543a3a4d6f6d656e7400004054536574207468652063757272656e742074696d652e005501546869732063616c6c2073686f756c6420626520696e766f6b65642065786163746c79206f6e63652070657220626c6f636b2e2049742077696c6c2070616e6963206174207468652066696e616c697a6174696f6ed470686173652c20696620746869732063616c6c206861736e2774206265656e20696e766f6b656420627920746861742074696d652e0041015468652074696d657374616d702073686f756c642062652067726561746572207468616e207468652070726576696f7573206f6e652062792074686520616d6f756e742073706563696669656420627940604d696e696d756d506572696f64602e00d4546865206469737061746368206f726967696e20666f7220746869732063616c6c206d7573742062652060496e686572656e74602e002823203c7765696768743e31012d20604f2831296020284e6f7465207468617420696d706c656d656e746174696f6e73206f6620604f6e54696d657374616d7053657460206d75737420616c736f20626520604f283129602961012d20312073746f72616765207265616420616e6420312073746f72616765206d75746174696f6e2028636f64656320604f28312960292e202862656361757365206f6620604469645570646174653a3a74616b656020696e402020606f6e5f66696e616c697a656029d42d2031206576656e742068616e646c657220606f6e5f74696d657374616d705f736574602e204d75737420626520604f283129602e2c23203c2f7765696768743e042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632e590100000620005d0110346672616d655f737570706f72741c73746f72616765407765616b5f626f756e6465645f766563385765616b426f756e646564566563080454016101045300000400690101185665633c543e00006101083c70616c6c65745f62616c616e6365732c42616c616e63654c6f636b041c42616c616e63650118000c01086964490101384c6f636b4964656e746966696572000118616d6f756e7418011c42616c616e636500011c726561736f6e736501011c526561736f6e7300006501083c70616c6c65745f62616c616e6365731c526561736f6e7300010c0c466565000000104d6973630001000c416c6c0002000069010000026101006d0110346672616d655f737570706f72741c73746f726167652c626f756e6465645f76656328426f756e646564566563080454017101045300000400750101185665633c543e00007101083c70616c6c65745f62616c616e6365732c52657365727665446174610844526573657276654964656e7469666965720149011c42616c616e6365011800080108696449010144526573657276654964656e746966696572000118616d6f756e7418011c42616c616e6365000075010000027101007901083c70616c6c65745f62616c616e6365732052656c65617365730001081856315f305f300000001856325f305f30000100007d010c3c70616c6c65745f62616c616e6365731870616c6c65741043616c6c080454000449000118207472616e73666572080110646573748101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f7572636500011476616c75658d010128543a3a42616c616e6365000064d45472616e7366657220736f6d65206c697175696420667265652062616c616e636520746f20616e6f74686572206163636f756e742e000501607472616e73666572602077696c6c207365742074686520604672656542616c616e636560206f66207468652073656e64657220616e642072656365697665722e11014966207468652073656e6465722773206163636f756e742069732062656c6f7720746865206578697374656e7469616c206465706f736974206173206120726573756c74b06f6620746865207472616e736665722c20746865206163636f756e742077696c6c206265207265617065642e001501546865206469737061746368206f726967696e20666f7220746869732063616c6c206d75737420626520605369676e65646020627920746865207472616e736163746f722e002823203c7765696768743e61012d20446570656e64656e74206f6e20617267756d656e747320627574206e6f7420637269746963616c2c20676976656e2070726f70657220696d706c656d656e746174696f6e7320666f7220696e70757420636f6e66696794202074797065732e205365652072656c617465642066756e6374696f6e732062656c6f772e31012d20497420636f6e7461696e732061206c696d69746564206e756d626572206f6620726561647320616e642077726974657320696e7465726e616c6c7920616e64206e6f20636f6d706c6578382020636f6d7075746174696f6e2e004852656c617465642066756e6374696f6e733a004d0120202d2060656e737572655f63616e5f77697468647261776020697320616c776179732063616c6c656420696e7465726e616c6c792062757420686173206120626f756e64656420636f6d706c65786974792e290120202d205472616e7366657272696e672062616c616e63657320746f206163636f756e7473207468617420646964206e6f74206578697374206265666f72652077696c6c206361757365cc2020202060543a3a4f6e4e65774163636f756e743a3a6f6e5f6e65775f6163636f756e746020746f2062652063616c6c65642e5d0120202d2052656d6f76696e6720656e6f7567682066756e64732066726f6d20616e206163636f756e742077696c6c20747269676765722060543a3a4475737452656d6f76616c3a3a6f6e5f756e62616c616e636564602e5d0120202d20607472616e736665725f6b6565705f616c6976656020776f726b73207468652073616d652077617920617320607472616e73666572602c206275742068617320616e206164646974696f6e616c20636865636bdc202020207468617420746865207472616e736665722077696c6c206e6f74206b696c6c20746865206f726967696e206163636f756e742e842d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d11012d204f726967696e206163636f756e7420697320616c726561647920696e206d656d6f72792c20736f206e6f204442206f7065726174696f6e7320666f72207468656d2e2c23203c2f7765696768743e2c7365745f62616c616e63650c010c77686f8101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f757263650001206e65775f667265658d010128543a3a42616c616e63650001306e65775f72657365727665648d010128543a3a42616c616e636500012090536574207468652062616c616e636573206f66206120676976656e206163636f756e742e001d01546869732077696c6c20616c74657220604672656542616c616e63656020616e642060526573657276656442616c616e63656020696e2073746f726167652e2069742077696c6c3101616c736f20616c7465722074686520746f74616c2069737375616e6365206f66207468652073797374656d202860546f74616c49737375616e6365602920617070726f7072696174656c792e1501496620746865206e65772066726565206f722072657365727665642062616c616e63652069732062656c6f7720746865206578697374656e7469616c206465706f7369742cfc69742077696c6c20726573657420746865206163636f756e74206e6f6e63652028606672616d655f73797374656d3a3a4163636f756e744e6f6e636560292e00b0546865206469737061746368206f726967696e20666f7220746869732063616c6c2069732060726f6f74602e38666f7263655f7472616e736665720c0118736f757263658101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f75726365000110646573748101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f7572636500011476616c75658d010128543a3a42616c616e63650002184d0145786163746c7920617320607472616e73666572602c2065786365707420746865206f726967696e206d75737420626520726f6f7420616e642074686520736f75726365206163636f756e74206d6179206265287370656369666965642e2823203c7765696768743e4d012d2053616d65206173207472616e736665722c20627574206164646974696f6e616c207265616420616e6420777269746520626563617573652074686520736f75726365206163636f756e74206973206e6f747c2020617373756d656420746f20626520696e20746865206f7665726c61792e2c23203c2f7765696768743e4c7472616e736665725f6b6565705f616c697665080110646573748101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f7572636500011476616c75658d010128543a3a42616c616e63650003184d0153616d6520617320746865205b607472616e73666572605d2063616c6c2c206275742077697468206120636865636b207468617420746865207472616e736665722077696c6c206e6f74206b696c6c207468653c6f726967696e206163636f756e742e00b8393925206f66207468652074696d6520796f752077616e74205b607472616e73666572605d20696e73746561642e00c05b607472616e73666572605d3a207374727563742e50616c6c65742e68746d6c236d6574686f642e7472616e73666572307472616e736665725f616c6c080110646573748101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f757263650001286b6565705f616c697665b00110626f6f6c00044405015472616e736665722074686520656e74697265207472616e7366657261626c652062616c616e63652066726f6d207468652063616c6c6572206163636f756e742e0059014e4f54453a20546869732066756e6374696f6e206f6e6c7920617474656d70747320746f207472616e73666572205f7472616e7366657261626c655f2062616c616e6365732e2054686973206d65616e7320746861746101616e79206c6f636b65642c2072657365727665642c206f72206578697374656e7469616c206465706f7369747320287768656e20606b6565705f616c6976656020697320607472756560292c2077696c6c206e6f742062655d017472616e7366657272656420627920746869732066756e6374696f6e2e20546f20656e73757265207468617420746869732066756e6374696f6e20726573756c747320696e2061206b696c6c6564206163636f756e742c4501796f75206d69676874206e65656420746f207072657061726520746865206163636f756e742062792072656d6f76696e6720616e79207265666572656e636520636f756e746572732c2073746f72616765406465706f736974732c206574632e2e2e00c0546865206469737061746368206f726967696e206f6620746869732063616c6c206d757374206265205369676e65642e00a02d206064657374603a2054686520726563697069656e74206f6620746865207472616e736665722e59012d20606b6565705f616c697665603a204120626f6f6c65616e20746f2064657465726d696e652069662074686520607472616e736665725f616c6c60206f7065726174696f6e2073686f756c642073656e6420616c6c4d0120206f66207468652066756e647320746865206163636f756e74206861732c2063617573696e67207468652073656e646572206163636f756e7420746f206265206b696c6c6564202866616c7365292c206f72590120207472616e736665722065766572797468696e6720657863657074206174206c6561737420746865206578697374656e7469616c206465706f7369742c2077686963682077696c6c2067756172616e74656520746fc820206b656570207468652073656e646572206163636f756e7420616c697665202874727565292e2023203c7765696768743e39012d204f2831292e204a757374206c696b65207472616e736665722c206275742072656164696e672074686520757365722773207472616e7366657261626c652062616c616e63652066697273742e302020233c2f7765696768743e3c666f7263655f756e7265736572766508010c77686f8101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f75726365000118616d6f756e74180128543a3a42616c616e636500050cb0556e7265736572766520736f6d652062616c616e63652066726f6d2061207573657220627920666f7263652e006c43616e206f6e6c792062652063616c6c656420627920524f4f542e042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632e81010c2873705f72756e74696d65306d756c746961646472657373304d756c74694164647265737308244163636f756e7449640100304163636f756e74496e6465780174011408496404000001244163636f756e74496400000014496e6465780400850101304163636f756e74496e6465780001000c526177040028011c5665633c75383e0002002441646472657373333204000401205b75383b2033325d000300244164647265737332300400890101205b75383b2032305d000400008501000006740089010000031400000008008d01000006180091010c3c70616c6c65745f62616c616e6365731870616c6c6574144572726f720804540004490001203856657374696e6742616c616e63650000049856657374696e672062616c616e636520746f6f206869676820746f2073656e642076616c7565544c69717569646974795265737472696374696f6e73000104c44163636f756e74206c6971756964697479207265737472696374696f6e732070726576656e74207769746864726177616c4c496e73756666696369656e7442616c616e63650002047442616c616e636520746f6f206c6f7720746f2073656e642076616c7565484578697374656e7469616c4465706f736974000304e856616c756520746f6f206c6f7720746f20637265617465206163636f756e742064756520746f206578697374656e7469616c206465706f736974244b656570416c6976650004048c5472616e736665722f7061796d656e7420776f756c64206b696c6c206163636f756e745c4578697374696e6756657374696e675363686564756c65000504c8412076657374696e67207363686564756c6520616c72656164792065786973747320666f722074686973206163636f756e742c446561644163636f756e740006048842656e6566696369617279206163636f756e74206d757374207072652d65786973743c546f6f4d616e795265736572766573000704ac4e756d626572206f66206e616d656420726573657276657320657863656564204d6178526573657276657304ad010a090909437573746f6d205b6469737061746368206572726f72735d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f7273290a0909096f6620746869732070616c6c65742e0a09090995010c3473705f61726974686d657469632c66697865645f706f696e7424466978656455313238000004001801107531323800009901086870616c6c65745f7472616e73616374696f6e5f7061796d656e742052656c6561736573000108245631416e6369656e74000000085632000100009d01000002a10100a1010c346672616d655f737570706f72741c7765696768747358576569676874546f466565436f656666696369656e74041c42616c616e6365011800100134636f6566665f696e746567657218011c42616c616e6365000128636f6566665f667261630d01011c50657262696c6c0001206e65676174697665b00110626f6f6c00011864656772656508010875380000a5010c2c70616c6c65745f7375646f1870616c6c65741043616c6c040454000110107375646f04011063616c6ca9010160426f783c3c5420617320436f6e6669673e3a3a43616c6c3e000028350141757468656e7469636174657320746865207375646f206b657920616e64206469737061746368657320612066756e6374696f6e2063616c6c20776974682060526f6f7460206f726967696e2e00cc546865206469737061746368206f726967696e20666f7220746869732063616c6c206d757374206265205f5369676e65645f2e002823203c7765696768743e1c2d204f2831292e602d204c696d697465642073746f726167652072656164732e5c2d204f6e6520444220777269746520286576656e74292ec42d20576569676874206f662064657269766174697665206063616c6c6020657865637574696f6e202b2031302c3030302e2c23203c2f7765696768743e547375646f5f756e636865636b65645f77656967687408011063616c6ca9010160426f783c3c5420617320436f6e6669673e3a3a43616c6c3e000118776569676874200118576569676874000128350141757468656e7469636174657320746865207375646f206b657920616e64206469737061746368657320612066756e6374696f6e2063616c6c20776974682060526f6f7460206f726967696e2e2d01546869732066756e6374696f6e20646f6573206e6f7420636865636b2074686520776569676874206f66207468652063616c6c2c20616e6420696e737465616420616c6c6f777320746865b05375646f207573657220746f20737065636966792074686520776569676874206f66207468652063616c6c2e00cc546865206469737061746368206f726967696e20666f7220746869732063616c6c206d757374206265205f5369676e65645f2e002823203c7765696768743e1c2d204f2831292ecc2d2054686520776569676874206f6620746869732063616c6c20697320646566696e6564206279207468652063616c6c65722e2c23203c2f7765696768743e1c7365745f6b657904010c6e65778101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f757263650002285d0141757468656e74696361746573207468652063757272656e74207375646f206b657920616e6420736574732074686520676976656e204163636f756e7449642028606e6577602920617320746865206e6577207375646f106b65792e00cc546865206469737061746368206f726967696e20666f7220746869732063616c6c206d757374206265205f5369676e65645f2e002823203c7765696768743e1c2d204f2831292e602d204c696d697465642073746f726167652072656164732e402d204f6e65204442206368616e67652e2c23203c2f7765696768743e1c7375646f5f617308010c77686f8101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f7572636500011063616c6ca9010160426f783c3c5420617320436f6e6669673e3a3a43616c6c3e00032c4d0141757468656e7469636174657320746865207375646f206b657920616e64206469737061746368657320612066756e6374696f6e2063616c6c207769746820605369676e656460206f726967696e2066726f6d406120676976656e206163636f756e742e00cc546865206469737061746368206f726967696e20666f7220746869732063616c6c206d757374206265205f5369676e65645f2e002823203c7765696768743e1c2d204f2831292e602d204c696d697465642073746f726167652072656164732e5c2d204f6e6520444220777269746520286576656e74292ec42d20576569676874206f662064657269766174697665206063616c6c6020657865637574696f6e202b2031302c3030302e2c23203c2f7765696768743e042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632ea901085c637265646974636f696e5f6e6f64655f72756e74696d651043616c6c0001181853797374656d0400090101ad0173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c53797374656d2c2052756e74696d653e0000002454696d657374616d700400550101b90173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c54696d657374616d702c2052756e74696d653e0001002042616c616e63657304007d0101b50173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c42616c616e6365732c2052756e74696d653e000200105375646f0400a50101a50173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c5375646f2c2052756e74696d653e00040028437265646974636f696e0400ad0101bd0173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c437265646974636f696e2c2052756e74696d653e00050028446966666963756c74790400dd0101bd0173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c446966666963756c74792c2052756e74696d653e00060000ad010c4470616c6c65745f637265646974636f696e1870616c6c65741043616c6c0404540001384c636c61696d5f6c65676163795f77616c6c65740401287075626c69635f6b6579b101015873705f636f72653a3a65636473613a3a5075626c69630000041d01436c61696d73206c65676163792077616c6c657420616e64207472616e7366657273207468652062616c616e636520746f207468652073656e6465722773206163636f756e742e4072656769737465725f61646472657373080128626c6f636b636861696e880128426c6f636b636861696e00011c616464726573738c013c45787465726e616c41646472657373000104410152656769737465727320616e2065787465726e616c2061646472657373206f6e2060626c6f636b636861696e6020616e6420606e6574776f726b6020776974682076616c756520606164647265737360346164645f61736b5f6f72646572100128616464726573735f69648001484164647265737349643c543a3a486173683e0001147465726d73c001504c6f616e5465726d733c543a3a4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e000110677569648c011047756964000200346164645f6269645f6f72646572100128616464726573735f69648001484164647265737349643c543a3a486173683e0001147465726d73c001504c6f616e5465726d733c543a3a4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e000110677569648c011047756964000300246164645f6f666665720c013061736b5f6f726465725f6964b4018c41736b4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0001306269645f6f726465725f6964c4018c4269644f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e000400386164645f6465616c5f6f726465720801206f666665725f6964d001804f6666657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e0005003c6c6f636b5f6465616c5f6f726465720401346465616c5f6f726465725f6964a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0006003c66756e645f6465616c5f6f726465720801346465616c5f6f726465725f6964a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00012c7472616e736665725f696490014c5472616e7366657249643c543a3a486173683e0007004c72656769737465725f6465616c5f6f726465722001446c656e6465725f616464726573735f69648001484164647265737349643c543a3a486173683e00014c626f72726f7765725f616464726573735f69648001484164647265737349643c543a3a486173683e0001147465726d73c001504c6f616e5465726d733c543a3a4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e00012061736b5f677569648c0110477569640001206269645f677569648c011047756964000130626f72726f7765725f6b6579b9010124543a3a5369676e6572000148626f72726f7765725f7369676e6174757265c5010148543a3a5369676e65725369676e617475726500080040636c6f73655f6465616c5f6f726465720801346465616c5f6f726465725f6964a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00012c7472616e736665725f696490014c5472616e7366657249643c543a3a486173683e0009004472656769737465725f7472616e736665721001347472616e736665725f6b696e649801305472616e736665724b696e640001106761696ea8013845787465726e616c416d6f756e740001206f726465725f69649c01804f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e000140626c6f636b636861696e5f74785f69648c013045787465726e616c54784964000a00186578656d70740801346465616c5f6f726465725f6964a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00012c7472616e736665725f696490014c5472616e7366657249643c543a3a486173683e000b003c7665726966795f7472616e736665720401207472616e736665729401bc5472616e736665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e000c00346164645f617574686f7269747904010c77686f000130543a3a4163636f756e744964000d00042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632eb1010c1c73705f636f7265146563647361185075626c696300000400b50101205b75383b2033335d0000b501000003210000000800b901082873705f72756e74696d652c4d756c74695369676e657200010c1c456432353531390400bd01013c656432353531393a3a5075626c69630000001c537232353531390400c101013c737232353531393a3a5075626c69630001001445636473610400b101013465636473613a3a5075626c696300020000bd010c1c73705f636f72651c65643235353139185075626c6963000004000401205b75383b2033325d0000c1010c1c73705f636f72651c73723235353139185075626c6963000004000401205b75383b2033325d0000c501082873705f72756e74696d65384d756c74695369676e617475726500010c1c456432353531390400c9010148656432353531393a3a5369676e61747572650000001c537232353531390400d1010148737232353531393a3a5369676e61747572650001001445636473610400d501014065636473613a3a5369676e617475726500020000c9010c1c73705f636f72651c65643235353139245369676e617475726500000400cd0101205b75383b2036345d0000cd01000003400000000800d1010c1c73705f636f72651c73723235353139245369676e617475726500000400cd0101205b75383b2036345d0000d5010c1c73705f636f7265146563647361245369676e617475726500000400d90101205b75383b2036355d0000d901000003410000000800dd010c4470616c6c65745f646966666963756c74791870616c6c65741043616c6c040454000108547365745f7461726765745f626c6f636b5f74696d6504012c7461726765745f74696d65200124543a3a4d6f6d656e74000000547365745f61646a7573746d656e745f706572696f64040118706572696f64e101010c693634000100042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632ee1010000050c00e5010c2c70616c6c65745f7375646f1870616c6c6574144572726f720404540001042c526571756972655375646f0000047c53656e646572206d75737420626520746865205375646f206163636f756e7404644572726f7220666f7220746865205375646f2070616c6c6574e90110346672616d655f737570706f72741c73746f726167652c626f756e6465645f76656328426f756e64656456656308045401ed01045300000400f10101185665633c543e0000ed010c4470616c6c65745f637265646974636f696e14747970657348556e76657269666965645472616e736665720c244163636f756e744964010020426c6f636b4e756d011010486173680124000c01207472616e7366657294018c5472616e736665723c4163636f756e7449642c20426c6f636b4e756d2c20486173683e00013466726f6d5f65787465726e616c8c013c45787465726e616c4164647265737300012c746f5f65787465726e616c8c013c45787465726e616c416464726573730000f101000002ed0100f50100000408102400f9010c4470616c6c65745f637265646974636f696e1870616c6c6574144572726f720404540001c46041646472657373416c72656164795265676973746572656400000411015468652073706563696669656420616464726573732068617320616c7265616479206265656e207265676973746572656420746f20616e6f74686572206163636f756e74484e6f6e4578697374656e7441646472657373000100504e6f6e4578697374656e744465616c4f726465720002004c4e6f6e4578697374656e7441736b4f726465720003004c4e6f6e4578697374656e744269644f72646572000400404e6f6e4578697374656e744f666665720005004c4e6f6e4578697374656e745472616e73666572000600645472616e73666572416c726561647952656769737465726564000700405472616e736665724d69736d61746368000800605472616e73666572416c726561647950726f636573736564000900685472616e73666572416d6f756e74496e73756666696369656e74000a00444d616c666f726d65645472616e73666572000b005c556e737570706f727465645472616e736665724b696e64000c0054496e73756666696369656e74417574686f72697479000d00644e6f6e4578697374656e7452657061796d656e744f72646572000e002c4475706c69636174654964000f003c4e6f74416464726573734f776e6572001000584f6666636861696e5369676e656454784661696c6564001100584e6f4c6f63616c41636374466f725369676e656454780012006452657061796d656e744f726465724e6f6e5a65726f4761696e0013005c41646472657373506c6174666f726d4d69736d6174636800140040416c7265616479417574686f72697479001500384475706c69636174654f66666572001600344465616c4e6f7446756e646564001700584465616c4f72646572416c726561647946756e646564001800584465616c4f72646572416c7265616479436c6f736564001900584465616c4f72646572416c72656164794c6f636b6564001a00544465616c4f726465724d75737442654c6f636b6564001b00484475706c69636174654465616c4f72646572001c00404465616c4f7264657245787069726564001d003c41736b4f7264657245787069726564001e003c4269644f7264657245787069726564001f00304f66666572457870697265640020003841736b4269644d69736d617463680021002453616d654f776e657200220040496e76616c69645369676e61747572650023002c4e6f74426f72726f776572002400484d616c666f726d65644465616c4f72646572002500244e6f744c656e646572002600405363616c654465636f64654572726f7200270068556e76657269666965645472616e73666572506f6f6c46756c6c0028006452657061796d656e744f72646572556e737570706f72746564002900504e6f744c656761637957616c6c65744f776e6572002a00584c6567616379536967686173684d616c666f726d6564002b00504c656761637957616c6c65744e6f74466f756e64002c00684c656761637942616c616e63654b65657065724d697373696e67002d004c566572696679537472696e67546f6f4c6f6e67002e003c47756964416c726561647955736564002f003c496e76616c69644d6174757269747900300004ad010a090909437573746f6d205b6469737061746368206572726f72735d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f7273290a0909096f6620746869732070616c6c65742e0a090909fd0110346672616d655f737570706f72741c73746f726167652c626f756e6465645f76656328426f756e646564566563080454010102045300000400050201185665633c543e00000102084470616c6c65745f646966666963756c747958446966666963756c7479416e6454696d657374616d7004184d6f6d656e74012000080128646966666963756c7479a80128446966666963756c747900012474696d657374616d702001184d6f6d656e740000050200000201020009020c4470616c6c65745f646966666963756c74791870616c6c6574144572726f7204045400010c385a65726f54617267657454696d65000000505a65726f41646a7573746d656e74506572696f64000100604e6567617469766541646a7573746d656e74506572696f6400020004ad010a090909437573746f6d205b6469737061746368206572726f72735d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f7273290a0909096f6620746869732070616c6c65742e0a0909090d02102873705f72756e74696d651c67656e657269634c756e636865636b65645f65787472696e73696348556e636865636b656445787472696e736963101c416464726573730181011043616c6c01a901245369676e617475726501c5011445787472610111020004002800000011020000041c150219021d02210229022d02310200150210306672616d655f73797374656d28657874656e73696f6e7348636865636b5f737065635f76657273696f6e40436865636b5370656356657273696f6e04045400000000190210306672616d655f73797374656d28657874656e73696f6e7340636865636b5f74785f76657273696f6e38436865636b547856657273696f6e040454000000001d0210306672616d655f73797374656d28657874656e73696f6e7334636865636b5f67656e6573697330436865636b47656e6573697304045400000000210210306672616d655f73797374656d28657874656e73696f6e733c636865636b5f6d6f7274616c69747938436865636b4d6f7274616c697479040454000004002502010c45726100002502102873705f72756e74696d651c67656e657269630c6572610c4572610001010420496d6d6f7274616c0000001c4d6f7274616c31040008000001001c4d6f7274616c32040008000002001c4d6f7274616c33040008000003001c4d6f7274616c34040008000004001c4d6f7274616c35040008000005001c4d6f7274616c36040008000006001c4d6f7274616c37040008000007001c4d6f7274616c38040008000008001c4d6f7274616c3904000800000900204d6f7274616c313004000800000a00204d6f7274616c313104000800000b00204d6f7274616c313204000800000c00204d6f7274616c313304000800000d00204d6f7274616c313404000800000e00204d6f7274616c313504000800000f00204d6f7274616c313604000800001000204d6f7274616c313704000800001100204d6f7274616c313804000800001200204d6f7274616c313904000800001300204d6f7274616c323004000800001400204d6f7274616c323104000800001500204d6f7274616c323204000800001600204d6f7274616c323304000800001700204d6f7274616c323404000800001800204d6f7274616c323504000800001900204d6f7274616c323604000800001a00204d6f7274616c323704000800001b00204d6f7274616c323804000800001c00204d6f7274616c323904000800001d00204d6f7274616c333004000800001e00204d6f7274616c333104000800001f00204d6f7274616c333204000800002000204d6f7274616c333304000800002100204d6f7274616c333404000800002200204d6f7274616c333504000800002300204d6f7274616c333604000800002400204d6f7274616c333704000800002500204d6f7274616c333804000800002600204d6f7274616c333904000800002700204d6f7274616c343004000800002800204d6f7274616c343104000800002900204d6f7274616c343204000800002a00204d6f7274616c343304000800002b00204d6f7274616c343404000800002c00204d6f7274616c343504000800002d00204d6f7274616c343604000800002e00204d6f7274616c343704000800002f00204d6f7274616c343804000800003000204d6f7274616c343904000800003100204d6f7274616c353004000800003200204d6f7274616c353104000800003300204d6f7274616c353204000800003400204d6f7274616c353304000800003500204d6f7274616c353404000800003600204d6f7274616c353504000800003700204d6f7274616c353604000800003800204d6f7274616c353704000800003900204d6f7274616c353804000800003a00204d6f7274616c353904000800003b00204d6f7274616c363004000800003c00204d6f7274616c363104000800003d00204d6f7274616c363204000800003e00204d6f7274616c363304000800003f00204d6f7274616c363404000800004000204d6f7274616c363504000800004100204d6f7274616c363604000800004200204d6f7274616c363704000800004300204d6f7274616c363804000800004400204d6f7274616c363904000800004500204d6f7274616c373004000800004600204d6f7274616c373104000800004700204d6f7274616c373204000800004800204d6f7274616c373304000800004900204d6f7274616c373404000800004a00204d6f7274616c373504000800004b00204d6f7274616c373604000800004c00204d6f7274616c373704000800004d00204d6f7274616c373804000800004e00204d6f7274616c373904000800004f00204d6f7274616c383004000800005000204d6f7274616c383104000800005100204d6f7274616c383204000800005200204d6f7274616c383304000800005300204d6f7274616c383404000800005400204d6f7274616c383504000800005500204d6f7274616c383604000800005600204d6f7274616c383704000800005700204d6f7274616c383804000800005800204d6f7274616c383904000800005900204d6f7274616c393004000800005a00204d6f7274616c393104000800005b00204d6f7274616c393204000800005c00204d6f7274616c393304000800005d00204d6f7274616c393404000800005e00204d6f7274616c393504000800005f00204d6f7274616c393604000800006000204d6f7274616c393704000800006100204d6f7274616c393804000800006200204d6f7274616c393904000800006300244d6f7274616c31303004000800006400244d6f7274616c31303104000800006500244d6f7274616c31303204000800006600244d6f7274616c31303304000800006700244d6f7274616c31303404000800006800244d6f7274616c31303504000800006900244d6f7274616c31303604000800006a00244d6f7274616c31303704000800006b00244d6f7274616c31303804000800006c00244d6f7274616c31303904000800006d00244d6f7274616c31313004000800006e00244d6f7274616c31313104000800006f00244d6f7274616c31313204000800007000244d6f7274616c31313304000800007100244d6f7274616c31313404000800007200244d6f7274616c31313504000800007300244d6f7274616c31313604000800007400244d6f7274616c31313704000800007500244d6f7274616c31313804000800007600244d6f7274616c31313904000800007700244d6f7274616c31323004000800007800244d6f7274616c31323104000800007900244d6f7274616c31323204000800007a00244d6f7274616c31323304000800007b00244d6f7274616c31323404000800007c00244d6f7274616c31323504000800007d00244d6f7274616c31323604000800007e00244d6f7274616c31323704000800007f00244d6f7274616c31323804000800008000244d6f7274616c31323904000800008100244d6f7274616c31333004000800008200244d6f7274616c31333104000800008300244d6f7274616c31333204000800008400244d6f7274616c31333304000800008500244d6f7274616c31333404000800008600244d6f7274616c31333504000800008700244d6f7274616c31333604000800008800244d6f7274616c31333704000800008900244d6f7274616c31333804000800008a00244d6f7274616c31333904000800008b00244d6f7274616c31343004000800008c00244d6f7274616c31343104000800008d00244d6f7274616c31343204000800008e00244d6f7274616c31343304000800008f00244d6f7274616c31343404000800009000244d6f7274616c31343504000800009100244d6f7274616c31343604000800009200244d6f7274616c31343704000800009300244d6f7274616c31343804000800009400244d6f7274616c31343904000800009500244d6f7274616c31353004000800009600244d6f7274616c31353104000800009700244d6f7274616c31353204000800009800244d6f7274616c31353304000800009900244d6f7274616c31353404000800009a00244d6f7274616c31353504000800009b00244d6f7274616c31353604000800009c00244d6f7274616c31353704000800009d00244d6f7274616c31353804000800009e00244d6f7274616c31353904000800009f00244d6f7274616c3136300400080000a000244d6f7274616c3136310400080000a100244d6f7274616c3136320400080000a200244d6f7274616c3136330400080000a300244d6f7274616c3136340400080000a400244d6f7274616c3136350400080000a500244d6f7274616c3136360400080000a600244d6f7274616c3136370400080000a700244d6f7274616c3136380400080000a800244d6f7274616c3136390400080000a900244d6f7274616c3137300400080000aa00244d6f7274616c3137310400080000ab00244d6f7274616c3137320400080000ac00244d6f7274616c3137330400080000ad00244d6f7274616c3137340400080000ae00244d6f7274616c3137350400080000af00244d6f7274616c3137360400080000b000244d6f7274616c3137370400080000b100244d6f7274616c3137380400080000b200244d6f7274616c3137390400080000b300244d6f7274616c3138300400080000b400244d6f7274616c3138310400080000b500244d6f7274616c3138320400080000b600244d6f7274616c3138330400080000b700244d6f7274616c3138340400080000b800244d6f7274616c3138350400080000b900244d6f7274616c3138360400080000ba00244d6f7274616c3138370400080000bb00244d6f7274616c3138380400080000bc00244d6f7274616c3138390400080000bd00244d6f7274616c3139300400080000be00244d6f7274616c3139310400080000bf00244d6f7274616c3139320400080000c000244d6f7274616c3139330400080000c100244d6f7274616c3139340400080000c200244d6f7274616c3139350400080000c300244d6f7274616c3139360400080000c400244d6f7274616c3139370400080000c500244d6f7274616c3139380400080000c600244d6f7274616c3139390400080000c700244d6f7274616c3230300400080000c800244d6f7274616c3230310400080000c900244d6f7274616c3230320400080000ca00244d6f7274616c3230330400080000cb00244d6f7274616c3230340400080000cc00244d6f7274616c3230350400080000cd00244d6f7274616c3230360400080000ce00244d6f7274616c3230370400080000cf00244d6f7274616c3230380400080000d000244d6f7274616c3230390400080000d100244d6f7274616c3231300400080000d200244d6f7274616c3231310400080000d300244d6f7274616c3231320400080000d400244d6f7274616c3231330400080000d500244d6f7274616c3231340400080000d600244d6f7274616c3231350400080000d700244d6f7274616c3231360400080000d800244d6f7274616c3231370400080000d900244d6f7274616c3231380400080000da00244d6f7274616c3231390400080000db00244d6f7274616c3232300400080000dc00244d6f7274616c3232310400080000dd00244d6f7274616c3232320400080000de00244d6f7274616c3232330400080000df00244d6f7274616c3232340400080000e000244d6f7274616c3232350400080000e100244d6f7274616c3232360400080000e200244d6f7274616c3232370400080000e300244d6f7274616c3232380400080000e400244d6f7274616c3232390400080000e500244d6f7274616c3233300400080000e600244d6f7274616c3233310400080000e700244d6f7274616c3233320400080000e800244d6f7274616c3233330400080000e900244d6f7274616c3233340400080000ea00244d6f7274616c3233350400080000eb00244d6f7274616c3233360400080000ec00244d6f7274616c3233370400080000ed00244d6f7274616c3233380400080000ee00244d6f7274616c3233390400080000ef00244d6f7274616c3234300400080000f000244d6f7274616c3234310400080000f100244d6f7274616c3234320400080000f200244d6f7274616c3234330400080000f300244d6f7274616c3234340400080000f400244d6f7274616c3234350400080000f500244d6f7274616c3234360400080000f600244d6f7274616c3234370400080000f700244d6f7274616c3234380400080000f800244d6f7274616c3234390400080000f900244d6f7274616c3235300400080000fa00244d6f7274616c3235310400080000fb00244d6f7274616c3235320400080000fc00244d6f7274616c3235330400080000fd00244d6f7274616c3235340400080000fe00244d6f7274616c3235350400080000ff0000290210306672616d655f73797374656d28657874656e73696f6e732c636865636b5f6e6f6e636528436865636b4e6f6e63650404540000040001010120543a3a496e64657800002d0210306672616d655f73797374656d28657874656e73696f6e7330636865636b5f7765696768742c436865636b576569676874040454000000003102086870616c6c65745f7472616e73616374696f6e5f7061796d656e74604368617267655472616e73616374696f6e5061796d656e74040454000004008d01013042616c616e63654f663c543e00003502085c637265646974636f696e5f6e6f64655f72756e74696d651c52756e74696d6500000000201853797374656d011853797374656d401c4163636f756e7401010402000c4101000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004e8205468652066756c6c206163636f756e7420696e666f726d6174696f6e20666f72206120706172746963756c6172206163636f756e742049442e3845787472696e736963436f756e74000010040004b820546f74616c2065787472696e7369637320636f756e7420666f72207468652063757272656e7420626c6f636b2e2c426c6f636b57656967687401001c600000000000000000000000000000000000000000000000000488205468652063757272656e742077656967687420666f722074686520626c6f636b2e40416c6c45787472696e736963734c656e000010040004410120546f74616c206c656e6774682028696e2062797465732920666f7220616c6c2065787472696e736963732070757420746f6765746865722c20666f72207468652063757272656e7420626c6f636b2e24426c6f636b486173680101040510248000000000000000000000000000000000000000000000000000000000000000000498204d6170206f6620626c6f636b206e756d6265727320746f20626c6f636b206861736865732e3445787472696e736963446174610101040510280400043d012045787472696e73696373206461746120666f72207468652063757272656e7420626c6f636b20286d61707320616e2065787472696e736963277320696e64657820746f206974732064617461292e184e756d6265720100101000000000040901205468652063757272656e7420626c6f636b206e756d626572206265696e672070726f6365737365642e205365742062792060657865637574655f626c6f636b602e28506172656e744861736801002480000000000000000000000000000000000000000000000000000000000000000004702048617368206f66207468652070726576696f757320626c6f636b2e1844696765737401002c040004f020446967657374206f66207468652063757272656e7420626c6f636b2c20616c736f2070617274206f662074686520626c6f636b206865616465722e184576656e747301003c040010a0204576656e7473206465706f736974656420666f72207468652063757272656e7420626c6f636b2e005901204e4f54453a20546869732073746f72616765206974656d206973206578706c696369746c7920756e626f756e6465642073696e6365206974206973206e6576657220696e74656e64656420746f2062652072656164642066726f6d2077697468696e207468652072756e74696d652e284576656e74436f756e74010010100000000004b820546865206e756d626572206f66206576656e747320696e2074686520604576656e74733c543e60206c6973742e2c4576656e74546f706963730101040224f40400282501204d617070696e67206265747765656e206120746f7069632028726570726573656e74656420627920543a3a486173682920616e64206120766563746f72206f6620696e646578657394206f66206576656e747320696e2074686520603c4576656e74733c543e3e60206c6973742e00510120416c6c20746f70696320766563746f727320686176652064657465726d696e69737469632073746f72616765206c6f636174696f6e7320646570656e64696e67206f6e2074686520746f7069632e2054686973450120616c6c6f7773206c696768742d636c69656e747320746f206c6576657261676520746865206368616e67657320747269652073746f7261676520747261636b696e67206d656368616e69736d20616e64e420696e2063617365206f66206368616e67657320666574636820746865206c697374206f66206576656e7473206f6620696e7465726573742e004d01205468652076616c756520686173207468652074797065206028543a3a426c6f636b4e756d6265722c204576656e74496e646578296020626563617573652069662077652075736564206f6e6c79206a7573744d012074686520604576656e74496e64657860207468656e20696e20636173652069662074686520746f70696320686173207468652073616d6520636f6e74656e7473206f6e20746865206e65787420626c6f636b0101206e6f206e6f74696669636174696f6e2077696c6c20626520747269676765726564207468757320746865206576656e74206d69676874206265206c6f73742e484c61737452756e74696d65557067726164650000fc04000455012053746f726573207468652060737065635f76657273696f6e6020616e642060737065635f6e616d6560206f66207768656e20746865206c6173742072756e74696d6520757067726164652068617070656e65642e545570677261646564546f553332526566436f756e740100b00400044d012054727565206966207765206861766520757067726164656420736f207468617420607479706520526566436f756e74602069732060753332602e2046616c7365202864656661756c7429206966206e6f742e605570677261646564546f547269706c65526566436f756e740100b00400085d012054727565206966207765206861766520757067726164656420736f2074686174204163636f756e74496e666f20636f6e7461696e73207468726565207479706573206f662060526566436f756e74602e2046616c736548202864656661756c7429206966206e6f742e38457865637574696f6e50686173650000ec040004882054686520657865637574696f6e207068617365206f662074686520626c6f636b2e01090101481830426c6f636b576569676874731d01850100f2052a010000000040e59c30120000405973070000000001c076eec4d20b00000100f0ab75a40d0000010000000000000000405973070000000001c0c627ec5e100000010040e59c3012000001005039278c040000405973070000000000000004d020426c6f636b20262065787472696e7369637320776569676874733a20626173652076616c75657320616e64206c696d6974732e2c426c6f636b4c656e6774682d013000003c00000050000000500004a820546865206d6178696d756d206c656e677468206f66206120626c6f636b2028696e206279746573292e38426c6f636b48617368436f756e74101060090000045501204d6178696d756d206e756d626572206f6620626c6f636b206e756d62657220746f20626c6f636b2068617368206d617070696e677320746f206b65657020286f6c64657374207072756e6564206669727374292e20446257656967687435014040787d010000000000e1f505000000000409012054686520776569676874206f662072756e74696d65206461746162617365206f7065726174696f6e73207468652072756e74696d652063616e20696e766f6b652e1c56657273696f6e3901a9023c637265646974636f696e2d6e6f64653c637265646974636f696e2d6e6f646501000000650000000100000028df6acb689907609b0400000037e397fc7c91f5e40100000040fe3ad401f8959a05000000d2bc9897eed08f1503000000f78b278be53f454c02000000ab3c0572291feb8b01000000bc9d89904f5b923f0100000037c8bb1350a9a2a8010000001ea173a1db199b3b010000006c7049b21e2444110100000001000000010484204765742074686520636861696e27732063757272656e742076657273696f6e2e28535335385072656669784d01082a0014a8205468652064657369676e61746564205353383520707265666978206f66207468697320636861696e2e0039012054686973207265706c6163657320746865202273733538466f726d6174222070726f7065727479206465636c6172656420696e2074686520636861696e20737065632e20526561736f6e20697331012074686174207468652072756e74696d652073686f756c64206b6e6f772061626f7574207468652070726566697820696e206f7264657220746f206d616b6520757365206f662069742061737020616e206964656e746966696572206f662074686520636861696e2e015101002454696d657374616d70012454696d657374616d70080c4e6f7701002020000000000000000004902043757272656e742074696d6520666f72207468652063757272656e7420626c6f636b2e244469645570646174650100b0040004b420446964207468652074696d657374616d7020676574207570646174656420696e207468697320626c6f636b3f0155010004344d696e696d756d506572696f642020e803000000000000104d0120546865206d696e696d756d20706572696f64206265747765656e20626c6f636b732e204265776172652074686174207468697320697320646966666572656e7420746f20746865202a65787065637465642a5d0120706572696f6420746861742074686520626c6f636b2070726f64756374696f6e206170706172617475732070726f76696465732e20596f75722063686f73656e20636f6e73656e7375732073797374656d2077696c6c5d012067656e6572616c6c7920776f726b2077697468207468697320746f2064657465726d696e6520612073656e7369626c6520626c6f636b2074696d652e20652e672e20466f7220417572612c2069742077696c6c206265a020646f75626c65207468697320706572696f64206f6e2064656661756c742073657474696e67732e00012042616c616e636573012042616c616e6365731434546f74616c49737375616e6365010018400000000000000000000000000000000004982054686520746f74616c20756e6974732069737375656420696e207468652073797374656d2e1c4163636f756e740101040200140101000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c6c205468652062616c616e6365206f6620616e206163636f756e742e004101204e4f54453a2054686973206973206f6e6c79207573656420696e207468652063617365207468617420746869732070616c6c6574206973207573656420746f2073746f72652062616c616e6365732e144c6f636b7301010402005d01040008b820416e79206c6971756964697479206c6f636b73206f6e20736f6d65206163636f756e742062616c616e6365732e2501204e4f54453a2053686f756c64206f6e6c79206265206163636573736564207768656e2073657474696e672c206368616e67696e6720616e642066726565696e672061206c6f636b2e20526573657276657301010402006d01040004a4204e616d6564207265736572766573206f6e20736f6d65206163636f756e742062616c616e6365732e3853746f7261676556657273696f6e0100790104000c7c2053746f726167652076657273696f6e206f66207468652070616c6c65742e00a020546869732069732073657420746f2076322e302e3020666f72206e6577206e6574776f726b732e017d0101640c484578697374656e7469616c4465706f7369741840f401000000000000000000000000000004d420546865206d696e696d756d20616d6f756e7420726571756972656420746f206b65657020616e206163636f756e74206f70656e2e204d61784c6f636b7310103200000008f420546865206d6178696d756d206e756d626572206f66206c6f636b7320746861742073686f756c64206578697374206f6e20616e206163636f756e742edc204e6f74207374726963746c7920656e666f726365642c20627574207573656420666f722077656967687420657374696d6174696f6e2e2c4d61785265736572766573101000000000040d0120546865206d6178696d756d206e756d626572206f66206e616d656420726573657276657320746861742063616e206578697374206f6e20616e206163636f756e742e01910102485472616e73616374696f6e5061796d656e7401485472616e73616374696f6e5061796d656e7408444e6578744665654d756c7469706c6965720100950140000064a7b3b6e00d0000000000000000003853746f7261676556657273696f6e0100990104000000000c485472616e73616374696f6e42797465466565184001000000000000000000000000000000040d01205468652066656520746f206265207061696420666f72206d616b696e672061207472616e73616374696f6e3b20746865207065722d6279746520706f7274696f6e2e604f7065726174696f6e616c4665654d756c7469706c696572080401545901204120666565206d756c6974706c69657220666f7220604f7065726174696f6e616c602065787472696e7369637320746f20636f6d7075746520227669727475616c207469702220746f20626f6f73742074686569722c20607072696f7269747960004d0120546869732076616c7565206973206d756c7469706c656420627920746865206066696e616c5f6665656020746f206f627461696e206120227669727475616c20746970222074686174206973206c61746572f420616464656420746f20612074697020636f6d706f6e656e7420696e20726567756c617220607072696f72697479602063616c63756c6174696f6e732e4d01204974206d65616e732074686174206120604e6f726d616c60207472616e73616374696f6e2063616e2066726f6e742d72756e20612073696d696c61726c792d73697a656420604f7065726174696f6e616c6041012065787472696e736963202877697468206e6f20746970292c20627920696e636c7564696e672061207469702076616c75652067726561746572207468616e20746865207669727475616c207469702e003c20606060727573742c69676e6f726540202f2f20466f7220604e6f726d616c608c206c6574207072696f72697479203d207072696f726974795f63616c6328746970293b0054202f2f20466f7220604f7065726174696f6e616c601101206c6574207669727475616c5f746970203d2028696e636c7573696f6e5f666565202b2074697029202a204f7065726174696f6e616c4665654d756c7469706c6965723bc4206c6574207072696f72697479203d207072696f726974795f63616c6328746970202b207669727475616c5f746970293b1020606060005101204e6f746520746861742073696e636520776520757365206066696e616c5f6665656020746865206d756c7469706c696572206170706c69657320616c736f20746f2074686520726567756c61722060746970605d012073656e74207769746820746865207472616e73616374696f6e2e20536f2c206e6f74206f6e6c7920646f657320746865207472616e73616374696f6e206765742061207072696f726974792062756d702062617365646101206f6e207468652060696e636c7573696f6e5f666565602c2062757420776520616c736f20616d706c6966792074686520696d70616374206f662074697073206170706c69656420746f20604f7065726174696f6e616c6038207472616e73616374696f6e732e2c576569676874546f4665659d015c04005a6202000000000000000000000000000000000001040d012054686520706f6c796e6f6d69616c2074686174206973206170706c69656420696e206f7264657220746f20646572697665206665652066726f6d207765696768742e0003105375646f01105375646f040c4b6579000000040004842054686520604163636f756e74496460206f6620746865207375646f206b65792e01a501016c0001e5010428437265646974636f696e0128437265646974636f696e2c2c417574686f726974696573000104020074040000344c656761637957616c6c65747300010403e0180400004c4c656761637942616c616e63654b65657065720000000400004c556e76657269666965645472616e73666572730100e901040000284465616c4f72646572730001080506f501d80400002441646472657373657300010402808404000024557365644775696473000104028c740400002441736b4f72646572730001080506f501b8040000244269644f72646572730001080506f501c8040000184f66666572730001080506f501d4040000245472616e736665727300010406909404000001ad01017c0001f9010528446966666963756c74790128446966666963756c7479108450726576696f7573446966666963756c74696573416e6454696d657374616d70730100fd010400004443757272656e74446966666963756c74790100a8800000000000000000000000000000000000000000000000000000000000000000003c546172676574426c6f636b54696d650100202000000000000000000068446966666963756c747941646a7573746d656e74506572696f640100e1012000000000000000000001dd010000010902061c52657761726473011c52657761726473042c426c6f636b417574686f720000000400000001e80000070d02041c40436865636b5370656356657273696f6e15021038436865636b547856657273696f6e19021030436865636b47656e657369731d022438436865636b4d6f7274616c69747921022428436865636b4e6f6e63652902742c436865636b5765696768742d0274604368617267655472616e73616374696f6e5061796d656e743102743502",
-    "id": "1"
-}
\ No newline at end of file
+{"jsonrpc":"2.0","result":"0x6d6574610e3902000c1c73705f636f72651863727970746f2c4163636f756e7449643332000004000401205b75383b2033325d0000040000032000000008000800000503000c08306672616d655f73797374656d2c4163636f756e74496e666f0814496e64657801102c4163636f756e74446174610114001401146e6f6e6365100114496e646578000124636f6e73756d657273100120526566436f756e7400012470726f766964657273100120526566436f756e7400012c73756666696369656e7473100120526566436f756e740001106461746114012c4163636f756e7444617461000010000005050014083c70616c6c65745f62616c616e6365732c4163636f756e7444617461041c42616c616e63650118001001106672656518011c42616c616e6365000120726573657276656418011c42616c616e636500012c6d6973635f66726f7a656e18011c42616c616e63650001286665655f66726f7a656e18011c42616c616e636500001800000507001c0c346672616d655f737570706f72741c77656967687473405065724469737061746368436c6173730404540120000c01186e6f726d616c2001045400012c6f7065726174696f6e616c200104540001246d616e6461746f727920010454000020000005060024083c7072696d69746976655f74797065731048323536000004000401205b75383b2033325d00002800000208002c102873705f72756e74696d651c67656e65726963186469676573741844696765737400000401106c6f677330013c5665633c4469676573744974656d3e000030000002340034102873705f72756e74696d651c67656e6572696318646967657374284469676573744974656d0001142850726552756e74696d650800380144436f6e73656e737573456e67696e654964000028011c5665633c75383e00060024436f6e73656e7375730800380144436f6e73656e737573456e67696e654964000028011c5665633c75383e000400105365616c0800380144436f6e73656e737573456e67696e654964000028011c5665633c75383e000500144f74686572040028011c5665633c75383e0000006452756e74696d65456e7669726f6e6d656e745570646174656400080000380000030400000008003c00000240004008306672616d655f73797374656d2c4576656e745265636f7264080445014404540124000c01147068617365ec011450686173650001146576656e7444010445000118746f70696373f001185665633c543e000044085c637265646974636f696e5f6e6f64655f72756e74696d65144576656e740001141853797374656d04004801706672616d655f73797374656d3a3a4576656e743c52756e74696d653e0000002042616c616e636573040064017c70616c6c65745f62616c616e6365733a3a4576656e743c52756e74696d653e000200105375646f04006c016c70616c6c65745f7375646f3a3a4576656e743c52756e74696d653e00040028437265646974636f696e04007c018470616c6c65745f637265646974636f696e3a3a4576656e743c52756e74696d653e0005001c526577617264730400e8017870616c6c65745f726577617264733a3a4576656e743c52756e74696d653e00070000480c306672616d655f73797374656d1870616c6c6574144576656e740404540001184045787472696e7369635375636365737304013464697370617463685f696e666f4c01304469737061746368496e666f00000490416e2065787472696e73696320636f6d706c65746564207375636365737366756c6c792e3c45787472696e7369634661696c656408013864697370617463685f6572726f7258013444697370617463684572726f7200013464697370617463685f696e666f4c01304469737061746368496e666f00010450416e2065787472696e736963206661696c65642e2c436f64655570646174656400020450603a636f6465602077617320757064617465642e284e65774163636f756e7404011c6163636f756e74000130543a3a4163636f756e7449640003046841206e6577206163636f756e742077617320637265617465642e344b696c6c65644163636f756e7404011c6163636f756e74000130543a3a4163636f756e74496400040458416e206163636f756e7420776173207265617065642e2052656d61726b656408011873656e646572000130543a3a4163636f756e7449640001106861736824011c543a3a48617368000504704f6e206f6e2d636861696e2072656d61726b2068617070656e65642e04704576656e7420666f72207468652053797374656d2070616c6c65742e4c0c346672616d655f737570706f72741c77656967687473304469737061746368496e666f00000c0118776569676874200118576569676874000114636c6173735001344469737061746368436c617373000120706179735f666565540110506179730000500c346672616d655f737570706f72741c77656967687473344469737061746368436c61737300010c184e6f726d616c0000002c4f7065726174696f6e616c000100244d616e6461746f727900020000540c346672616d655f737570706f72741c7765696768747310506179730001080c596573000000084e6f0001000058082873705f72756e74696d653444697370617463684572726f72000124144f746865720000003043616e6e6f744c6f6f6b7570000100244261644f726967696e000200184d6f64756c65080114696e64657808010875380001146572726f72080108753800030044436f6e73756d657252656d61696e696e670004002c4e6f50726f76696465727300050040546f6f4d616e79436f6e73756d65727300060014546f6b656e04005c0128546f6b656e4572726f720007002841726974686d65746963040060013c41726974686d657469634572726f72000800005c082873705f72756e74696d6528546f6b656e4572726f7200011c1c4e6f46756e647300000020576f756c644469650001003042656c6f774d696e696d756d0002003043616e6e6f7443726561746500030030556e6b6e6f776e41737365740004001846726f7a656e0005002c556e737570706f727465640006000060082873705f72756e74696d653c41726974686d657469634572726f7200010c24556e646572666c6f77000000204f766572666c6f77000100384469766973696f6e42795a65726f00020000640c3c70616c6c65745f62616c616e6365731870616c6c6574144576656e740804540004490001281c456e646f77656408011c6163636f756e74000130543a3a4163636f756e744964000130667265655f62616c616e6365180128543a3a42616c616e6365000004b8416e206163636f756e74207761732063726561746564207769746820736f6d6520667265652062616c616e63652e20447573744c6f737408011c6163636f756e74000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e63650001083d01416e206163636f756e74207761732072656d6f7665642077686f73652062616c616e636520776173206e6f6e2d7a65726f206275742062656c6f77204578697374656e7469616c4465706f7369742c78726573756c74696e6720696e20616e206f75747269676874206c6f73732e205472616e736665720c011066726f6d000130543a3a4163636f756e744964000108746f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e63650002044c5472616e73666572207375636365656465642e2842616c616e63655365740c010c77686f000130543a3a4163636f756e74496400011066726565180128543a3a42616c616e63650001207265736572766564180128543a3a42616c616e636500030468412062616c616e6365207761732073657420627920726f6f742e20526573657276656408010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e6365000404e0536f6d652062616c616e63652077617320726573657276656420286d6f7665642066726f6d206672656520746f207265736572766564292e28556e726573657276656408010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e6365000504e8536f6d652062616c616e63652077617320756e726573657276656420286d6f7665642066726f6d20726573657276656420746f2066726565292e4852657365727665526570617472696174656410011066726f6d000130543a3a4163636f756e744964000108746f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e636500014864657374696e6174696f6e5f7374617475736801185374617475730006084d01536f6d652062616c616e636520776173206d6f7665642066726f6d207468652072657365727665206f6620746865206669727374206163636f756e7420746f20746865207365636f6e64206163636f756e742ed846696e616c20617267756d656e7420696e64696361746573207468652064657374696e6174696f6e2062616c616e636520747970652e1c4465706f73697408010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e6365000704d8536f6d6520616d6f756e7420776173206465706f73697465642028652e672e20666f72207472616e73616374696f6e2066656573292e20576974686472617708010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e63650008041d01536f6d6520616d6f756e74207761732077697468647261776e2066726f6d20746865206163636f756e742028652e672e20666f72207472616e73616374696f6e2066656573292e1c536c617368656408010c77686f000130543a3a4163636f756e744964000118616d6f756e74180128543a3a42616c616e63650009040101536f6d6520616d6f756e74207761732072656d6f7665642066726f6d20746865206163636f756e742028652e672e20666f72206d69736265686176696f72292e0499010a090909546865205b6576656e745d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f72732920656d69747465640a090909627920746869732070616c6c65742e0a0909096814346672616d655f737570706f72741874726169747318746f6b656e73106d6973633442616c616e63655374617475730001081046726565000000205265736572766564000100006c0c2c70616c6c65745f7375646f1870616c6c6574144576656e7404045400010c14537564696404012c7375646f5f726573756c747001384469737061746368526573756c740000048841207375646f206a75737420746f6f6b20706c6163652e205c5b726573756c745c5d284b65794368616e6765640401286f6c645f7375646f65727801504f7074696f6e3c543a3a4163636f756e7449643e0001043901546865205c5b7375646f65725c5d206a757374207377697463686564206964656e746974793b20746865206f6c64206b657920697320737570706c696564206966206f6e6520657869737465642e285375646f4173446f6e6504012c7375646f5f726573756c747001384469737061746368526573756c740002048841207375646f206a75737420746f6f6b20706c6163652e205c5b726573756c745c5d0499010a090909546865205b6576656e745d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f72732920656d69747465640a090909627920746869732070616c6c65742e0a090909700418526573756c740804540174044501580108084f6b040074000000000c45727204005800000100007400000400007804184f7074696f6e04045401000108104e6f6e6500000010536f6d6504000000000100007c0c4470616c6c65745f637265646974636f696e1870616c6c6574144576656e7404045400013044416464726573735265676973746572656408008001484164647265737349643c543a3a486173683e0000840154416464726573733c543a3a4163636f756e7449643e000008d0416e2061646472657373206f6e20616e2065787465726e616c20636861696e20686173206265656e20726567697374657265642eac5b726567697374657265645f616464726573735f69642c20726567697374657265645f616464726573735d485472616e7366657252656769737465726564080090014c5472616e7366657249643c543a3a486173683e00009401bc5472616e736665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e000108f8416e2065787465726e616c207472616e7366657220686173206265656e207265676973746572656420616e642077696c6c2062652076657269666965642eb45b726567697374657265645f7472616e736665725f69642c20726567697374657265645f7472616e736665725d405472616e736665725665726966696564080090014c5472616e7366657249643c543a3a486173683e00009401bc5472616e736665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e000208d0416e2065787465726e616c207472616e7366657220686173206265656e207375636365737366756c6c792076657269666965642ea45b76657269666965645f7472616e736665725f69642c2076657269666965645f7472616e736665725d445472616e7366657250726f636573736564080090014c5472616e7366657249643c543a3a486173683e00009401bc5472616e736665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e0003081501416e2065787465726e616c207472616e7366657220686173206265656e2070726f63657373656420616e64206d61726b65642061732070617274206f662061206c6f616e2eac5b70726f6365737365645f7472616e736665725f69642c2070726f6365737365645f7472616e736665725d3441736b4f7264657241646465640800b4018c41736b4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000b801e841736b4f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e00040c4d01416e2061736b206f7264657220686173206265656e20616464656420627920612070726f7370656374697665206c656e6465722e205468697320696e64696361746573207468617420746865206c656e646572b86973206c6f6f6b696e6720746f2069737375652061206c6f616e2077697468206365727461696e207465726d732e645b61736b5f6f726465725f69642c2061736b5f6f726465725d344269644f7264657241646465640800c4018c4269644f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000c801e84269644f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e00050c59014120626964206f7264657220686173206265656e20616464656420627920612070726f737065637469766520626f72726f7765722e205468697320696e6469636174657320746861742074686520626f72726f776572a46973206c6f6f6b696e6720666f722061206c6f616e2077697468206365727461696e207465726d732e645b6269645f6f726465725f69642c206269645f6f726465725d284f6666657241646465640800d001804f6666657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000d401b04f666665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e00060c0d01416e206f6666657220686173206265656e2061646465642062792061206c656e6465722e205468697320696e64696361746573207468617420746865206c656e6465720501697320696e746572657374656420696e20656e746572696e672061206c6f616e207769746820746865206f776e6572206f662074686520626964206f726465722e445b6f666665725f69642c206f666665725d384465616c4f7264657241646465640800a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000d801ec4465616c4f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e00070c2d0141206465616c206f7264657220686173206265656e206164646564206279206120626f72726f7765722e205468697320696e6469636174657320746861742074686520626f72726f776572f06861732061636365707465642061206c656e6465722773206f6666657220616e6420696e74656e647320746f20656e74657220746865206c6f616e2e6c5b6465616c5f6f726465725f69642c206465616c5f6f726465725d3c4465616c4f7264657246756e6465640800a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000d801ec4465616c4f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e000810210141206465616c206f7264657220686173206265656e2066756e6465642062792061206c656e6465722e205468697320696e64696361746573207468617420746865206c656e646572350168617320696e69746961746564207468652061637475616c206c6f616e206279207472616e7366657272696e6720746865206c6f616e20616d6f756e7420746f2074686520626f72726f776572546f6e20616e2065787465726e616c20636861696e2ea45b66756e6465645f6465616c5f6f726465725f69642c2066756e6465645f6465616c5f6f726465725d3c4465616c4f72646572436c6f7365640800a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0000d801ec4465616c4f726465723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173682c20543a3a4d6f6d656e743e00090c310141206465616c206f7264657220686173206265656e20636c6f736564206279206120626f72726f7765722e205468697320696e6469636174657320746861742074686520626f72726f776572f06861732072657061696420746865206c6f616e20696e2066756c6c20616e64206973206e6f7720636c6f73696e67206f757420746865206c6f616e2ea45b636c6f7365645f6465616c5f6f726465725f69642c20636c6f7365645f6465616c5f6f726465725d304c6f616e4578656d707465640800a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e000090014c5472616e7366657249643c543a3a486173683e000a10350141206c6f616e206578656d7074696f6e20686173206265656e206772616e7465642062792061206c656e6465722e205468697320696e64696361746573207468617420746865206c656e646572290169732072656c656173696e6720736f6d65206f7220616c6c206f6620746865206f75747374616e64696e672064656274206f6e20746865206c6f616e2e2054686520626f72726f776572c46973206e6f206c6f6e67657220726573706f6e7369626c6520666f72207265706179696e672074686520616d6f756e742ebc5b6578656d707465645f6465616c5f6f726465725f69642c206578656d7074696e675f7472616e736665725f69645d4c4c656761637957616c6c6574436c61696d65640c00000130543a3a4163636f756e7449640000e001344c6567616379536967686173680000180128543a3a42616c616e6365000b0c590141206c65676163792077616c6c65742066726f6d20437265646974636f696e20312e5820686173206265656e20636c61696d65642e205468652062616c616e6365206f6620746865206c65676163792077616c6c6574ec686173206265656e207472616e7366657272656420746f20746865206f776e6572277320437265646974636f696e20322e30206163636f756e742e15015b6c65676163795f77616c6c65745f636c61696d65722c206c65676163795f77616c6c65745f736967686173682c206c65676163795f77616c6c65745f62616c616e63655d0499010a090909546865205b6576656e745d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f72732920656d69747465640a090909627920746869732070616c6c65742e0a090909800c4470616c6c65745f637265646974636f696e147479706573244164647265737349640410486173680124000400240110486173680000840c4470616c6c65745f637265646974636f696e1474797065731c4164647265737304244163636f756e7449640100000c0128626c6f636b636861696e880128426c6f636b636861696e00011476616c75658c013c45787465726e616c416464726573730001146f776e65720001244163636f756e7449640000880c4470616c6c65745f637265646974636f696e14747970657328426c6f636b636861696e00011420457468657265756d0000001c52696e6b656279000100244c756e6976657273650002001c426974636f696e000300144f7468657204008c01284f74686572436861696e000400008c10346672616d655f737570706f72741c73746f726167652c626f756e6465645f76656328426f756e64656456656308045401080453000004002801185665633c543e0000900c4470616c6c65745f637265646974636f696e147479706573285472616e7366657249640410486173680124000400240110486173680000940c4470616c6c65745f637265646974636f696e147479706573205472616e736665720c244163636f756e744964010020426c6f636b4e756d01101048617368012400280128626c6f636b636861696e880128426c6f636b636861696e0001106b696e649801305472616e736665724b696e6400011066726f6d80013c4164647265737349643c486173683e000108746f80013c4164647265737349643c486173683e0001206f726465725f69649c015c4f7264657249643c426c6f636b4e756d2c20486173683e000118616d6f756e74a8013845787465726e616c416d6f756e7400010874788c013045787465726e616c54784964000114626c6f636b100120426c6f636b4e756d00012470726f636573736564b00110626f6f6c00011c736967686173680001244163636f756e7449640000980c4470616c6c65745f637265646974636f696e147479706573305472616e736665724b696e6400011014457263323004008c013c45787465726e616c416464726573730000001c4574686c65737304008c013c45787465726e616c41646472657373000100184e6174697665000200144f7468657204008c01444f746865725472616e736665724b696e64000300009c0c4470616c6c65745f637265646974636f696e1474797065731c4f7264657249640820426c6f636b4e756d0110104861736801240108104465616c0400a0016c4465616c4f7264657249643c426c6f636b4e756d2c20486173683e0000002452657061796d656e740400a4018052657061796d656e744f7264657249643c426c6f636b4e756d2c20486173683e00010000a00c4470616c6c65745f637265646974636f696e1474797065732c4465616c4f7264657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000a40c4470616c6c65745f637265646974636f696e1474797065734052657061796d656e744f7264657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000a8083c7072696d69746976655f7479706573105532353600000400ac01205b7536343b20345d0000ac000003040000002000b00000050000b40c4470616c6c65745f637265646974636f696e1474797065732841736b4f7264657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000b80c4470616c6c65745f637265646974636f696e1474797065732041736b4f7264657210244163636f756e744964010020426c6f636b4e756d011010486173680124184d6f6d656e74012000180128626c6f636b636861696e880128426c6f636b636861696e0001446c656e6465725f616464726573735f696480013c4164647265737349643c486173683e0001147465726d73bc014041736b5465726d733c4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100120426c6f636b4e756d000114626c6f636b100120426c6f636b4e756d0001186c656e6465720001244163636f756e7449640000bc104470616c6c65745f637265646974636f696e147479706573286c6f616e5f7465726d732041736b5465726d7304184d6f6d656e740120000400c001444c6f616e5465726d733c4d6f6d656e743e0000c0104470616c6c65745f637265646974636f696e147479706573286c6f616e5f7465726d73244c6f616e5465726d7304184d6f6d656e740120000c0118616d6f756e74a8013845787465726e616c416d6f756e74000134696e7465726573745f72617465200130496e746572657374526174650001206d617475726974792001184d6f6d656e740000c40c4470616c6c65745f637265646974636f696e147479706573284269644f7264657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000c80c4470616c6c65745f637265646974636f696e147479706573204269644f7264657210244163636f756e744964010020426c6f636b4e756d011010486173680124184d6f6d656e74012000180128626c6f636b636861696e880128426c6f636b636861696e00014c626f72726f7765725f616464726573735f696480013c4164647265737349643c486173683e0001147465726d73cc01404269645465726d733c4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100120426c6f636b4e756d000114626c6f636b100120426c6f636b4e756d000120626f72726f7765720001244163636f756e7449640000cc104470616c6c65745f637265646974636f696e147479706573286c6f616e5f7465726d73204269645465726d7304184d6f6d656e740120000400c001444c6f616e5465726d733c4d6f6d656e743e0000d00c4470616c6c65745f637265646974636f696e1474797065731c4f6666657249640820426c6f636b4e756d011010486173680124000800100120426c6f636b4e756d0000240110486173680000d40c4470616c6c65745f637265646974636f696e147479706573144f666665720c244163636f756e744964010020426c6f636b4e756d01101048617368012400180128626c6f636b636861696e880128426c6f636b636861696e00011861736b5f6964b4016841736b4f7264657249643c426c6f636b4e756d2c20486173683e0001186269645f6964c401684269644f7264657249643c426c6f636b4e756d2c20486173683e00014065787069726174696f6e5f626c6f636b100120426c6f636b4e756d000114626c6f636b100120426c6f636b4e756d0001186c656e6465720001244163636f756e7449640000d80c4470616c6c65745f637265646974636f696e147479706573244465616c4f7264657210244163636f756e744964010020426c6f636b4e756d011010486173680124184d6f6d656e740120002c0128626c6f636b636861696e880128426c6f636b636861696e0001206f666665725f6964d0015c4f6666657249643c426c6f636b4e756d2c20486173683e0001446c656e6465725f616464726573735f696480013c4164647265737349643c486173683e00014c626f72726f7765725f616464726573735f696480013c4164647265737349643c486173683e0001147465726d73c001444c6f616e5465726d733c4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100120426c6f636b4e756d00012474696d657374616d702001184d6f6d656e7400014c66756e64696e675f7472616e736665725f6964dc01604f7074696f6e3c5472616e7366657249643c486173683e3e00015472657061796d656e745f7472616e736665725f6964dc01604f7074696f6e3c5472616e7366657249643c486173683e3e0001106c6f636b7801444f7074696f6e3c4163636f756e7449643e000120626f72726f7765720001244163636f756e7449640000dc04184f7074696f6e04045401900108104e6f6e6500000010536f6d650400900000010000e00c4470616c6c65745f637265646974636f696e147479706573344c65676163795369676861736800000400e401205b75383b2036305d0000e40000033c0000000800e80c3870616c6c65745f726577617264731870616c6c6574144576656e740404540001043052657761726449737375656408000001384163636f756e7449644f663c543e000018013042616c616e63654f663c543e000004a452657761726420776173206973737565642e205b626c6f636b5f617574686f722c20616d6f756e745d0499010a090909546865205b6576656e745d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f72732920656d69747465640a090909627920746869732070616c6c65742e0a090909ec08306672616d655f73797374656d14506861736500010c384170706c7945787472696e736963040010010c7533320000003046696e616c697a6174696f6e00010038496e697469616c697a6174696f6e00020000f00000022400f4000002f800f800000408101000fc08306672616d655f73797374656d584c61737452756e74696d6555706772616465496e666f0000080130737065635f76657273696f6e0101014c636f6465633a3a436f6d706163743c7533323e000124737065635f6e616d650501016473705f72756e74696d653a3a52756e74696d65537472696e670000010100000610000501000005020009010c306672616d655f73797374656d1870616c6c65741043616c6c0404540001242866696c6c5f626c6f636b040114726174696f0d01011c50657262696c6c00000405014120646973706174636820746861742077696c6c2066696c6c2074686520626c6f636b2077656967687420757020746f2074686520676976656e20726174696f2e1872656d61726b04011872656d61726b28011c5665633c75383e000114684d616b6520736f6d65206f6e2d636861696e2072656d61726b2e002823203c7765696768743e202d20604f283129602c23203c2f7765696768743e387365745f686561705f7061676573040114706167657320010c753634000204f853657420746865206e756d626572206f6620706167657320696e2074686520576562417373656d626c7920656e7669726f6e6d656e74277320686561702e207365745f636f6465040110636f646528011c5665633c75383e0003306453657420746865206e65772072756e74696d6520636f64652e002823203c7765696768743e31012d20604f2843202b2053296020776865726520604360206c656e677468206f662060636f64656020616e642060536020636f6d706c6578697479206f66206063616e5f7365745f636f64656045012d20312063616c6c20746f206063616e5f7365745f636f6465603a20604f28532960202863616c6c73206073705f696f3a3a6d6973633a3a72756e74696d655f76657273696f6e60207768696368206973342020657870656e73697665292e842d20312073746f726167652077726974652028636f64656320604f28432960292e402d203120646967657374206974656d2e282d2031206576656e742e4d0154686520776569676874206f6620746869732066756e6374696f6e20697320646570656e64656e74206f6e207468652072756e74696d652c206275742067656e6572616c6c7920746869732069732076657279b8657870656e736976652e2057652077696c6c207472656174207468697320617320612066756c6c20626c6f636b2e2c23203c2f7765696768743e5c7365745f636f64655f776974686f75745f636865636b73040110636f646528011c5665633c75383e000424190153657420746865206e65772072756e74696d6520636f646520776974686f757420646f696e6720616e7920636865636b73206f662074686520676976656e2060636f6465602e002823203c7765696768743e8c2d20604f2843296020776865726520604360206c656e677468206f662060636f646560842d20312073746f726167652077726974652028636f64656320604f28432960292e402d203120646967657374206974656d2e282d2031206576656e742e550154686520776569676874206f6620746869732066756e6374696f6e20697320646570656e64656e74206f6e207468652072756e74696d652e2057652077696c6c207472656174207468697320617320612066756c6c48626c6f636b2e2023203c2f7765696768743e2c7365745f73746f726167650401146974656d73110101345665633c4b657956616c75653e0005046853657420736f6d65206974656d73206f662073746f726167652e306b696c6c5f73746f726167650401106b657973190101205665633c4b65793e000604744b696c6c20736f6d65206974656d732066726f6d2073746f726167652e2c6b696c6c5f70726566697808011870726566697828010c4b657900011c7375626b65797310010c75333200071011014b696c6c20616c6c2073746f72616765206974656d7320776974682061206b657920746861742073746172747320776974682074686520676976656e207072656669782e0039012a2a4e4f54453a2a2a2057652072656c79206f6e2074686520526f6f74206f726967696e20746f2070726f7669646520757320746865206e756d626572206f66207375626b65797320756e6465723d0174686520707265666978207765206172652072656d6f76696e6720746f2061636375726174656c792063616c63756c6174652074686520776569676874206f6620746869732066756e6374696f6e2e4472656d61726b5f776974685f6576656e7404011872656d61726b28011c5665633c75383e000818a44d616b6520736f6d65206f6e2d636861696e2072656d61726b20616e6420656d6974206576656e742e002823203c7765696768743eb42d20604f28622960207768657265206220697320746865206c656e677468206f66207468652072656d61726b2e282d2031206576656e742e2c23203c2f7765696768743e042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632e0d010c3473705f61726974686d65746963287065725f7468696e67731c50657262696c6c0000040010010c75333200001101000002150100150100000408282800190100000228001d010c306672616d655f73797374656d186c696d69747330426c6f636b5765696768747300000c0128626173655f626c6f636b2001185765696768740001246d61785f626c6f636b2001185765696768740001247065725f636c617373210101845065724469737061746368436c6173733c57656967687473506572436c6173733e000021010c346672616d655f737570706f72741c77656967687473405065724469737061746368436c617373040454012501000c01186e6f726d616c250101045400012c6f7065726174696f6e616c25010104540001246d616e6461746f72792501010454000025010c306672616d655f73797374656d186c696d6974733c57656967687473506572436c6173730000100138626173655f65787472696e7369632001185765696768740001346d61785f65787472696e736963290101384f7074696f6e3c5765696768743e0001246d61785f746f74616c290101384f7074696f6e3c5765696768743e0001207265736572766564290101384f7074696f6e3c5765696768743e0000290104184f7074696f6e04045401200108104e6f6e6500000010536f6d6504002000000100002d010c306672616d655f73797374656d186c696d6974732c426c6f636b4c656e677468000004010c6d6178310101545065724469737061746368436c6173733c7533323e000031010c346672616d655f737570706f72741c77656967687473405065724469737061746368436c6173730404540110000c01186e6f726d616c1001045400012c6f7065726174696f6e616c100104540001246d616e6461746f727910010454000035010c346672616d655f737570706f72741c776569676874733c52756e74696d654462576569676874000008011072656164200118576569676874000114777269746520011857656967687400003901082873705f76657273696f6e3852756e74696d6556657273696f6e0000200124737065635f6e616d650501013452756e74696d65537472696e67000124696d706c5f6e616d650501013452756e74696d65537472696e67000144617574686f72696e675f76657273696f6e10010c753332000130737065635f76657273696f6e10010c753332000130696d706c5f76657273696f6e10010c753332000110617069733d01011c4170697356656300014c7472616e73616374696f6e5f76657273696f6e10010c75333200013473746174655f76657273696f6e080108753800003d01040c436f77040454014101000400410100000041010000024501004501000004084901100049010000030800000008004d01000005040051010c306672616d655f73797374656d1870616c6c6574144572726f720404540001183c496e76616c6964537065634e616d650000081101546865206e616d65206f662073706563696669636174696f6e20646f6573206e6f74206d61746368206265747765656e207468652063757272656e742072756e74696d6550616e6420746865206e65772072756e74696d652e685370656356657273696f6e4e65656473546f496e63726561736500010841015468652073706563696669636174696f6e2076657273696f6e206973206e6f7420616c6c6f77656420746f206465637265617365206265747765656e207468652063757272656e742072756e74696d6550616e6420746865206e65772072756e74696d652e744661696c6564546f4578747261637452756e74696d6556657273696f6e00020cec4661696c656420746f2065787472616374207468652072756e74696d652076657273696f6e2066726f6d20746865206e65772072756e74696d652e0009014569746865722063616c6c696e672060436f72655f76657273696f6e60206f72206465636f64696e67206052756e74696d6556657273696f6e60206661696c65642e4c4e6f6e44656661756c74436f6d706f73697465000304fc537569636964652063616c6c6564207768656e20746865206163636f756e7420686173206e6f6e2d64656661756c7420636f6d706f7369746520646174612e3c4e6f6e5a65726f526566436f756e74000404350154686572652069732061206e6f6e2d7a65726f207265666572656e636520636f756e742070726576656e74696e6720746865206163636f756e742066726f6d206265696e67207075726765642e3043616c6c46696c7465726564000504d0546865206f726967696e2066696c7465722070726576656e74207468652063616c6c20746f20626520646973706174636865642e046c4572726f7220666f72207468652053797374656d2070616c6c657455010c4070616c6c65745f74696d657374616d701870616c6c65741043616c6c0404540001040c73657404010c6e6f7759010124543a3a4d6f6d656e7400004054536574207468652063757272656e742074696d652e005501546869732063616c6c2073686f756c6420626520696e766f6b65642065786163746c79206f6e63652070657220626c6f636b2e2049742077696c6c2070616e6963206174207468652066696e616c697a6174696f6ed470686173652c20696620746869732063616c6c206861736e2774206265656e20696e766f6b656420627920746861742074696d652e0041015468652074696d657374616d702073686f756c642062652067726561746572207468616e207468652070726576696f7573206f6e652062792074686520616d6f756e742073706563696669656420627940604d696e696d756d506572696f64602e00d4546865206469737061746368206f726967696e20666f7220746869732063616c6c206d7573742062652060496e686572656e74602e002823203c7765696768743e31012d20604f2831296020284e6f7465207468617420696d706c656d656e746174696f6e73206f6620604f6e54696d657374616d7053657460206d75737420616c736f20626520604f283129602961012d20312073746f72616765207265616420616e6420312073746f72616765206d75746174696f6e2028636f64656320604f28312960292e202862656361757365206f6620604469645570646174653a3a74616b656020696e402020606f6e5f66696e616c697a656029d42d2031206576656e742068616e646c657220606f6e5f74696d657374616d705f736574602e204d75737420626520604f283129602e2c23203c2f7765696768743e042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632e590100000620005d0110346672616d655f737570706f72741c73746f72616765407765616b5f626f756e6465645f766563385765616b426f756e646564566563080454016101045300000400690101185665633c543e00006101083c70616c6c65745f62616c616e6365732c42616c616e63654c6f636b041c42616c616e63650118000c01086964490101384c6f636b4964656e746966696572000118616d6f756e7418011c42616c616e636500011c726561736f6e736501011c526561736f6e7300006501083c70616c6c65745f62616c616e6365731c526561736f6e7300010c0c466565000000104d6973630001000c416c6c0002000069010000026101006d0110346672616d655f737570706f72741c73746f726167652c626f756e6465645f76656328426f756e646564566563080454017101045300000400750101185665633c543e00007101083c70616c6c65745f62616c616e6365732c52657365727665446174610844526573657276654964656e7469666965720149011c42616c616e6365011800080108696449010144526573657276654964656e746966696572000118616d6f756e7418011c42616c616e6365000075010000027101007901083c70616c6c65745f62616c616e6365732052656c65617365730001081856315f305f300000001856325f305f30000100007d010c3c70616c6c65745f62616c616e6365731870616c6c65741043616c6c080454000449000118207472616e73666572080110646573748101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f7572636500011476616c75658d010128543a3a42616c616e6365000064d45472616e7366657220736f6d65206c697175696420667265652062616c616e636520746f20616e6f74686572206163636f756e742e000501607472616e73666572602077696c6c207365742074686520604672656542616c616e636560206f66207468652073656e64657220616e642072656365697665722e11014966207468652073656e6465722773206163636f756e742069732062656c6f7720746865206578697374656e7469616c206465706f736974206173206120726573756c74b06f6620746865207472616e736665722c20746865206163636f756e742077696c6c206265207265617065642e001501546865206469737061746368206f726967696e20666f7220746869732063616c6c206d75737420626520605369676e65646020627920746865207472616e736163746f722e002823203c7765696768743e61012d20446570656e64656e74206f6e20617267756d656e747320627574206e6f7420637269746963616c2c20676976656e2070726f70657220696d706c656d656e746174696f6e7320666f7220696e70757420636f6e66696794202074797065732e205365652072656c617465642066756e6374696f6e732062656c6f772e31012d20497420636f6e7461696e732061206c696d69746564206e756d626572206f6620726561647320616e642077726974657320696e7465726e616c6c7920616e64206e6f20636f6d706c6578382020636f6d7075746174696f6e2e004852656c617465642066756e6374696f6e733a004d0120202d2060656e737572655f63616e5f77697468647261776020697320616c776179732063616c6c656420696e7465726e616c6c792062757420686173206120626f756e64656420636f6d706c65786974792e290120202d205472616e7366657272696e672062616c616e63657320746f206163636f756e7473207468617420646964206e6f74206578697374206265666f72652077696c6c206361757365cc2020202060543a3a4f6e4e65774163636f756e743a3a6f6e5f6e65775f6163636f756e746020746f2062652063616c6c65642e5d0120202d2052656d6f76696e6720656e6f7567682066756e64732066726f6d20616e206163636f756e742077696c6c20747269676765722060543a3a4475737452656d6f76616c3a3a6f6e5f756e62616c616e636564602e5d0120202d20607472616e736665725f6b6565705f616c6976656020776f726b73207468652073616d652077617920617320607472616e73666572602c206275742068617320616e206164646974696f6e616c20636865636bdc202020207468617420746865207472616e736665722077696c6c206e6f74206b696c6c20746865206f726967696e206163636f756e742e842d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d11012d204f726967696e206163636f756e7420697320616c726561647920696e206d656d6f72792c20736f206e6f204442206f7065726174696f6e7320666f72207468656d2e2c23203c2f7765696768743e2c7365745f62616c616e63650c010c77686f8101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f757263650001206e65775f667265658d010128543a3a42616c616e63650001306e65775f72657365727665648d010128543a3a42616c616e636500012090536574207468652062616c616e636573206f66206120676976656e206163636f756e742e001d01546869732077696c6c20616c74657220604672656542616c616e63656020616e642060526573657276656442616c616e63656020696e2073746f726167652e2069742077696c6c3101616c736f20616c7465722074686520746f74616c2069737375616e6365206f66207468652073797374656d202860546f74616c49737375616e6365602920617070726f7072696174656c792e1501496620746865206e65772066726565206f722072657365727665642062616c616e63652069732062656c6f7720746865206578697374656e7469616c206465706f7369742cfc69742077696c6c20726573657420746865206163636f756e74206e6f6e63652028606672616d655f73797374656d3a3a4163636f756e744e6f6e636560292e00b0546865206469737061746368206f726967696e20666f7220746869732063616c6c2069732060726f6f74602e38666f7263655f7472616e736665720c0118736f757263658101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f75726365000110646573748101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f7572636500011476616c75658d010128543a3a42616c616e63650002184d0145786163746c7920617320607472616e73666572602c2065786365707420746865206f726967696e206d75737420626520726f6f7420616e642074686520736f75726365206163636f756e74206d6179206265287370656369666965642e2823203c7765696768743e4d012d2053616d65206173207472616e736665722c20627574206164646974696f6e616c207265616420616e6420777269746520626563617573652074686520736f75726365206163636f756e74206973206e6f747c2020617373756d656420746f20626520696e20746865206f7665726c61792e2c23203c2f7765696768743e4c7472616e736665725f6b6565705f616c697665080110646573748101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f7572636500011476616c75658d010128543a3a42616c616e63650003184d0153616d6520617320746865205b607472616e73666572605d2063616c6c2c206275742077697468206120636865636b207468617420746865207472616e736665722077696c6c206e6f74206b696c6c207468653c6f726967696e206163636f756e742e00b8393925206f66207468652074696d6520796f752077616e74205b607472616e73666572605d20696e73746561642e00c05b607472616e73666572605d3a207374727563742e50616c6c65742e68746d6c236d6574686f642e7472616e73666572307472616e736665725f616c6c080110646573748101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f757263650001286b6565705f616c697665b00110626f6f6c00044405015472616e736665722074686520656e74697265207472616e7366657261626c652062616c616e63652066726f6d207468652063616c6c6572206163636f756e742e0059014e4f54453a20546869732066756e6374696f6e206f6e6c7920617474656d70747320746f207472616e73666572205f7472616e7366657261626c655f2062616c616e6365732e2054686973206d65616e7320746861746101616e79206c6f636b65642c2072657365727665642c206f72206578697374656e7469616c206465706f7369747320287768656e20606b6565705f616c6976656020697320607472756560292c2077696c6c206e6f742062655d017472616e7366657272656420627920746869732066756e6374696f6e2e20546f20656e73757265207468617420746869732066756e6374696f6e20726573756c747320696e2061206b696c6c6564206163636f756e742c4501796f75206d69676874206e65656420746f207072657061726520746865206163636f756e742062792072656d6f76696e6720616e79207265666572656e636520636f756e746572732c2073746f72616765406465706f736974732c206574632e2e2e00c0546865206469737061746368206f726967696e206f6620746869732063616c6c206d757374206265205369676e65642e00a02d206064657374603a2054686520726563697069656e74206f6620746865207472616e736665722e59012d20606b6565705f616c697665603a204120626f6f6c65616e20746f2064657465726d696e652069662074686520607472616e736665725f616c6c60206f7065726174696f6e2073686f756c642073656e6420616c6c4d0120206f66207468652066756e647320746865206163636f756e74206861732c2063617573696e67207468652073656e646572206163636f756e7420746f206265206b696c6c6564202866616c7365292c206f72590120207472616e736665722065766572797468696e6720657863657074206174206c6561737420746865206578697374656e7469616c206465706f7369742c2077686963682077696c6c2067756172616e74656520746fc820206b656570207468652073656e646572206163636f756e7420616c697665202874727565292e2023203c7765696768743e39012d204f2831292e204a757374206c696b65207472616e736665722c206275742072656164696e672074686520757365722773207472616e7366657261626c652062616c616e63652066697273742e302020233c2f7765696768743e3c666f7263655f756e7265736572766508010c77686f8101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f75726365000118616d6f756e74180128543a3a42616c616e636500050cb0556e7265736572766520736f6d652062616c616e63652066726f6d2061207573657220627920666f7263652e006c43616e206f6e6c792062652063616c6c656420627920524f4f542e042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632e81010c2873705f72756e74696d65306d756c746961646472657373304d756c74694164647265737308244163636f756e7449640100304163636f756e74496e6465780174011408496404000001244163636f756e74496400000014496e6465780400850101304163636f756e74496e6465780001000c526177040028011c5665633c75383e0002002441646472657373333204000401205b75383b2033325d000300244164647265737332300400890101205b75383b2032305d000400008501000006740089010000031400000008008d01000006180091010c3c70616c6c65745f62616c616e6365731870616c6c6574144572726f720804540004490001203856657374696e6742616c616e63650000049856657374696e672062616c616e636520746f6f206869676820746f2073656e642076616c7565544c69717569646974795265737472696374696f6e73000104c44163636f756e74206c6971756964697479207265737472696374696f6e732070726576656e74207769746864726177616c4c496e73756666696369656e7442616c616e63650002047442616c616e636520746f6f206c6f7720746f2073656e642076616c7565484578697374656e7469616c4465706f736974000304e856616c756520746f6f206c6f7720746f20637265617465206163636f756e742064756520746f206578697374656e7469616c206465706f736974244b656570416c6976650004048c5472616e736665722f7061796d656e7420776f756c64206b696c6c206163636f756e745c4578697374696e6756657374696e675363686564756c65000504c8412076657374696e67207363686564756c6520616c72656164792065786973747320666f722074686973206163636f756e742c446561644163636f756e740006048842656e6566696369617279206163636f756e74206d757374207072652d65786973743c546f6f4d616e795265736572766573000704ac4e756d626572206f66206e616d656420726573657276657320657863656564204d6178526573657276657304ad010a090909437573746f6d205b6469737061746368206572726f72735d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f7273290a0909096f6620746869732070616c6c65742e0a09090995010c3473705f61726974686d657469632c66697865645f706f696e7424466978656455313238000004001801107531323800009901086870616c6c65745f7472616e73616374696f6e5f7061796d656e742052656c6561736573000108245631416e6369656e74000000085632000100009d01000002a10100a1010c346672616d655f737570706f72741c7765696768747358576569676874546f466565436f656666696369656e74041c42616c616e6365011800100134636f6566665f696e746567657218011c42616c616e6365000128636f6566665f667261630d01011c50657262696c6c0001206e65676174697665b00110626f6f6c00011864656772656508010875380000a5010c2c70616c6c65745f7375646f1870616c6c65741043616c6c040454000110107375646f04011063616c6ca9010160426f783c3c5420617320436f6e6669673e3a3a43616c6c3e000028350141757468656e7469636174657320746865207375646f206b657920616e64206469737061746368657320612066756e6374696f6e2063616c6c20776974682060526f6f7460206f726967696e2e00cc546865206469737061746368206f726967696e20666f7220746869732063616c6c206d757374206265205f5369676e65645f2e002823203c7765696768743e1c2d204f2831292e602d204c696d697465642073746f726167652072656164732e5c2d204f6e6520444220777269746520286576656e74292ec42d20576569676874206f662064657269766174697665206063616c6c6020657865637574696f6e202b2031302c3030302e2c23203c2f7765696768743e547375646f5f756e636865636b65645f77656967687408011063616c6ca9010160426f783c3c5420617320436f6e6669673e3a3a43616c6c3e000118776569676874200118576569676874000128350141757468656e7469636174657320746865207375646f206b657920616e64206469737061746368657320612066756e6374696f6e2063616c6c20776974682060526f6f7460206f726967696e2e2d01546869732066756e6374696f6e20646f6573206e6f7420636865636b2074686520776569676874206f66207468652063616c6c2c20616e6420696e737465616420616c6c6f777320746865b05375646f207573657220746f20737065636966792074686520776569676874206f66207468652063616c6c2e00cc546865206469737061746368206f726967696e20666f7220746869732063616c6c206d757374206265205f5369676e65645f2e002823203c7765696768743e1c2d204f2831292ecc2d2054686520776569676874206f6620746869732063616c6c20697320646566696e6564206279207468652063616c6c65722e2c23203c2f7765696768743e1c7365745f6b657904010c6e65778101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f757263650002285d0141757468656e74696361746573207468652063757272656e74207375646f206b657920616e6420736574732074686520676976656e204163636f756e7449642028606e6577602920617320746865206e6577207375646f106b65792e00cc546865206469737061746368206f726967696e20666f7220746869732063616c6c206d757374206265205f5369676e65645f2e002823203c7765696768743e1c2d204f2831292e602d204c696d697465642073746f726167652072656164732e402d204f6e65204442206368616e67652e2c23203c2f7765696768743e1c7375646f5f617308010c77686f8101018c3c543a3a4c6f6f6b7570206173205374617469634c6f6f6b75703e3a3a536f7572636500011063616c6ca9010160426f783c3c5420617320436f6e6669673e3a3a43616c6c3e00032c4d0141757468656e7469636174657320746865207375646f206b657920616e64206469737061746368657320612066756e6374696f6e2063616c6c207769746820605369676e656460206f726967696e2066726f6d406120676976656e206163636f756e742e00cc546865206469737061746368206f726967696e20666f7220746869732063616c6c206d757374206265205f5369676e65645f2e002823203c7765696768743e1c2d204f2831292e602d204c696d697465642073746f726167652072656164732e5c2d204f6e6520444220777269746520286576656e74292ec42d20576569676874206f662064657269766174697665206063616c6c6020657865637574696f6e202b2031302c3030302e2c23203c2f7765696768743e042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632ea901085c637265646974636f696e5f6e6f64655f72756e74696d651043616c6c0001181853797374656d0400090101ad0173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c53797374656d2c2052756e74696d653e0000002454696d657374616d700400550101b90173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c54696d657374616d702c2052756e74696d653e0001002042616c616e63657304007d0101b50173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c42616c616e6365732c2052756e74696d653e000200105375646f0400a50101a50173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c5375646f2c2052756e74696d653e00040028437265646974636f696e0400ad0101bd0173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c437265646974636f696e2c2052756e74696d653e00050028446966666963756c74790400dd0101bd0173656c663a3a73705f6170695f68696464656e5f696e636c756465735f636f6e7374727563745f72756e74696d653a3a68696464656e5f696e636c7564653a3a64697370617463680a3a3a43616c6c61626c6543616c6c466f723c446966666963756c74792c2052756e74696d653e00060000ad010c4470616c6c65745f637265646974636f696e1870616c6c65741043616c6c0404540001384c636c61696d5f6c65676163795f77616c6c65740401287075626c69635f6b6579b101015873705f636f72653a3a65636473613a3a5075626c69630000041d01436c61696d73206c65676163792077616c6c657420616e64207472616e7366657273207468652062616c616e636520746f207468652073656e6465722773206163636f756e742e4072656769737465725f61646472657373080128626c6f636b636861696e880128426c6f636b636861696e00011c616464726573738c013c45787465726e616c41646472657373000104410152656769737465727320616e2065787465726e616c2061646472657373206f6e2060626c6f636b636861696e6020616e6420606e6574776f726b6020776974682076616c756520606164647265737360346164645f61736b5f6f72646572100128616464726573735f69648001484164647265737349643c543a3a486173683e0001147465726d73c001504c6f616e5465726d733c543a3a4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e000110677569648c011047756964000200346164645f6269645f6f72646572100128616464726573735f69648001484164647265737349643c543a3a486173683e0001147465726d73c001504c6f616e5465726d733c543a3a4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e000110677569648c011047756964000300246164645f6f666665720c013061736b5f6f726465725f6964b4018c41736b4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0001306269645f6f726465725f6964c4018c4269644f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e000400386164645f6465616c5f6f726465720801206f666665725f6964d001804f6666657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e0005003c6c6f636b5f6465616c5f6f726465720401346465616c5f6f726465725f6964a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e0006003c66756e645f6465616c5f6f726465720801346465616c5f6f726465725f6964a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00012c7472616e736665725f696490014c5472616e7366657249643c543a3a486173683e0007004c72656769737465725f6465616c5f6f726465722001446c656e6465725f616464726573735f69648001484164647265737349643c543a3a486173683e00014c626f72726f7765725f616464726573735f69648001484164647265737349643c543a3a486173683e0001147465726d73c001504c6f616e5465726d733c543a3a4d6f6d656e743e00014065787069726174696f6e5f626c6f636b100144426c6f636b4e756d626572466f723c543e00012061736b5f677569648c0110477569640001206269645f677569648c011047756964000130626f72726f7765725f6b6579b9010124543a3a5369676e6572000148626f72726f7765725f7369676e6174757265c5010148543a3a5369676e65725369676e617475726500080040636c6f73655f6465616c5f6f726465720801346465616c5f6f726465725f6964a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00012c7472616e736665725f696490014c5472616e7366657249643c543a3a486173683e0009004472656769737465725f7472616e736665721001347472616e736665725f6b696e649801305472616e736665724b696e640001106761696ea8013845787465726e616c416d6f756e740001206f726465725f69649c01804f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e000140626c6f636b636861696e5f74785f69648c013045787465726e616c54784964000a00186578656d70740801346465616c5f6f726465725f6964a001904465616c4f7264657249643c543a3a426c6f636b4e756d6265722c20543a3a486173683e00012c7472616e736665725f696490014c5472616e7366657249643c543a3a486173683e000b003c7665726966795f7472616e736665720401207472616e736665729401bc5472616e736665723c543a3a4163636f756e7449642c20543a3a426c6f636b4e756d6265722c20543a3a486173683e000c00346164645f617574686f7269747904010c77686f000130543a3a4163636f756e744964000d00042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632eb1010c1c73705f636f7265146563647361185075626c696300000400b50101205b75383b2033335d0000b501000003210000000800b901082873705f72756e74696d652c4d756c74695369676e657200010c1c456432353531390400bd01013c656432353531393a3a5075626c69630000001c537232353531390400c101013c737232353531393a3a5075626c69630001001445636473610400b101013465636473613a3a5075626c696300020000bd010c1c73705f636f72651c65643235353139185075626c6963000004000401205b75383b2033325d0000c1010c1c73705f636f72651c73723235353139185075626c6963000004000401205b75383b2033325d0000c501082873705f72756e74696d65384d756c74695369676e617475726500010c1c456432353531390400c9010148656432353531393a3a5369676e61747572650000001c537232353531390400d1010148737232353531393a3a5369676e61747572650001001445636473610400d501014065636473613a3a5369676e617475726500020000c9010c1c73705f636f72651c65643235353139245369676e617475726500000400cd0101205b75383b2036345d0000cd01000003400000000800d1010c1c73705f636f72651c73723235353139245369676e617475726500000400cd0101205b75383b2036345d0000d5010c1c73705f636f7265146563647361245369676e617475726500000400d90101205b75383b2036355d0000d901000003410000000800dd010c4470616c6c65745f646966666963756c74791870616c6c65741043616c6c040454000108547365745f7461726765745f626c6f636b5f74696d6504012c7461726765745f74696d65200124543a3a4d6f6d656e74000000547365745f61646a7573746d656e745f706572696f64040118706572696f64e101010c693634000100042501436f6e7461696e73206f6e652076617269616e742070657220646973706174636861626c6520746861742063616e2062652063616c6c656420627920616e2065787472696e7369632ee1010000050c00e5010c2c70616c6c65745f7375646f1870616c6c6574144572726f720404540001042c526571756972655375646f0000047c53656e646572206d75737420626520746865205375646f206163636f756e7404644572726f7220666f7220746865205375646f2070616c6c6574e90110346672616d655f737570706f72741c73746f726167652c626f756e6465645f76656328426f756e64656456656308045401ed01045300000400f10101185665633c543e0000ed010c4470616c6c65745f637265646974636f696e14747970657348556e76657269666965645472616e736665720c244163636f756e744964010020426c6f636b4e756d011010486173680124000c01207472616e7366657294018c5472616e736665723c4163636f756e7449642c20426c6f636b4e756d2c20486173683e00013466726f6d5f65787465726e616c8c013c45787465726e616c4164647265737300012c746f5f65787465726e616c8c013c45787465726e616c416464726573730000f101000002ed0100f50100000408102400f9010c4470616c6c65745f637265646974636f696e1870616c6c6574144572726f720404540001b46041646472657373416c72656164795265676973746572656400000411015468652073706563696669656420616464726573732068617320616c7265616479206265656e207265676973746572656420746f20616e6f74686572206163636f756e74484e6f6e4578697374656e74416464726573730001049454686520737065636966696564206164647265737320646f6573206e6f742065786973742e504e6f6e4578697374656e744465616c4f72646572000204a054686520737065636966696564206465616c206f7264657220646f6573206e6f742065786973742e4c4e6f6e4578697374656e7441736b4f726465720003049c546865207370656369666965642061736b206f7264657220646f6573206e6f742065786973742e4c4e6f6e4578697374656e744269644f726465720004049c5468652073706563696669656420626964206f7264657220646f6573206e6f742065786973742e404e6f6e4578697374656e744f666665720005048c54686520737065636966696564206f6666657220646f6573206e6f742065786973742e4c4e6f6e4578697374656e745472616e736665720006049854686520737065636966696564207472616e7366657220646f6573206e6f742065786973742e645472616e73666572416c726561647952656769737465726564000704a4546865207472616e736665722068617320616c7265616479206265656e20726567697374657265642e405472616e736665724d69736d61746368000808b4546865206163636f756e742074686174207265676973746572656420746865207472616e7366657220646f6573d46e6f74206d6174636820746865206163636f756e7420617474656d7074696e6720746f2075736520746865207472616e736665722e605472616e73666572416c726561647950726f636573736564000904ec546865207472616e736665722068617320616c7265616479206265656e2070726f63657373656420616e642063616e6e6f7420626520757365642e685472616e73666572416d6f756e74496e73756666696369656e74000a04f8546865207472616e7366657220616d6f756e74206973206c657373207468616e2074686520616d6f756e7420696e20746865206c6f616e207465726d732e444d616c666f726d65645472616e73666572000b080501546865207472616e73666572206973206d616c666f726d656420616e6420686173206120626c6f636b206e756d6265722067726561746572207468616e207468657c7469702e205468697320697320616e20696e7465726e616c206572726f722e5c556e737570706f727465645472616e736665724b696e64000c08e454686520737065636966696564207472616e736665722074797065206973206e6f742063757272656e746c7920737570706f727465642062799c74686520626c6f636b636861696e20746865206c6f616e206973206578656375746564206f6e2e54496e73756666696369656e74417574686f72697479000d040501546865206e6f646520646f6573206e6f7420686176652073756666696369656e7420617574686f7269747920746f207665726966792061207472616e736665722e2c4475706c69636174654964000e049c546865207370656369666965642049442068617320616c7265616479206265656e20757365642e3c4e6f74416464726573734f776e6572000f04f054686520616464726573732063616e6e6f742062652075736564206265636175736520746865207573657220646f6573206e6f74206f776e2069742e584f6666636861696e5369676e656454784661696c6564001008fc4661696c656420746f2073656e6420616e206f6666636861696e2063616c6c6261636b207472616e73616374696f6e2e2054686973206973206c696b656c7948616e20696e7465726e616c206572726f722e584e6f4c6f63616c41636374466f725369676e65645478001108f0546865206e6f646520697320616e20617574686f7269747920627574207468657265206973206e6f206163636f756e7420746f206372656174652061dc63616c6c6261636b207472616e73616374696f6e2e2054686973206973206c696b656c7920616e20696e7465726e616c206572726f722e6452657061796d656e744f726465724e6f6e5a65726f4761696e0012005c41646472657373506c6174666f726d4d69736d61746368001304f8546865206164647265737365732073706563696669656420617265206e6f74206f6e20636f6d70617469626c652065787465726e616c20636861696e732e40416c7265616479417574686f7269747900140490546865206163636f756e7420697320616c726561647920616e20617574686f726974792e384475706c69636174654f6666657200150480546865206f666665722068617320616c7265616479206265656e206d6164652e344465616c4e6f7446756e646564001604dc546865206465616c2063616e6e6f74206265206c6f636b65642062656361757365206974206973206e6f742066756e646564207965742e584465616c4f72646572416c726561647946756e646564001704f0546865206465616c206f7264657220697320616c72656164792066756e64656420616e642063616e6e6f742062652066756e64656420616761696e2e584465616c4f72646572416c7265616479436c6f736564001804f0546865206465616c206f7264657220697320616c726561647920636c6f73656420616e642063616e6e6f7420626520636c6f73656420616761696e2e584465616c4f72646572416c72656164794c6f636b6564001904f0546865206465616c206f7264657220697320616c7265616479206c6f636b656420616e642063616e6e6f74206265206c6f636b656420616761696e2e544465616c4f726465724d75737442654c6f636b6564001a04d8546865206465616c206f72646572206d757374206265206c6f636b6564206265666f72652069742063616e20626520636c6f7365642e484475706c69636174654465616c4f72646572001b0478546865206465616c206f7264657220616c7265616479206578697374732e404465616c4f7264657245787069726564001c04c8546865206465616c206f7264657220686173206578706972656420616e64206973206e6f206c6f6e6765722076616c69642e3c41736b4f7264657245787069726564001d04c45468652061736b206f7264657220686173206578706972656420616e64206973206e6f206c6f6e6765722076616c69642e3c4269644f7264657245787069726564001e04c454686520626964206f7264657220686173206578706972656420616e64206973206e6f206c6f6e6765722076616c69642e304f6666657245787069726564001f04cc546865206f66666572206f7264657220686173206578706972656420616e64206973206e6f206c6f6e6765722076616c69642e3841736b4269644d69736d61746368002004c0546865207465726d73206f66207468652061736b20616e6420626964206f7264657220646f206e6f742061677265652e2453616d654f776e65720021040d0154686520626964206f72646572206973206f776e65642062792074686520757365722c206120757365722063616e6e6f74206c656e6420746f207468656d73656c662e40496e76616c69645369676e6174757265002204e0546865207369676e617475726520646f6573206e6f74206d6174636820746865207075626c6963206b657920616e64206d6573736167652e2c4e6f74426f72726f776572002304a44f6e6c792074686520626f72726f7765722063616e20706572666f726d2074686520616374696f6e2e484d616c666f726d65644465616c4f726465720024080d01546865206465616c206f72646572206973206d616c666f726d656420616e6420686173206120626c6f636b206e756d6265722067726561746572207468616e207468657c7469702e205468697320697320616e20696e7465726e616c206572726f722e244e6f744c656e6465720025049c4f6e6c7920746865206c656e6465722063616e20706572666f726d2074686520616374696f6e2e68556e76657269666965645472616e73666572506f6f6c46756c6c002604e4546865207175657565206f6620756e7665726966696564207472616e73666572732069732066756c6c20666f72207468697320626c6f636b2e6452657061796d656e744f72646572556e737570706f72746564002704b452657061796d656e74206f726465727320617265206e6f742063757272656e746c7920737570706f727465642e504e6f744c656761637957616c6c65744f776e6572002804ac546865206c65676163792077616c6c6574206973206e6f74206f776e65642062792074686520757365722e504c656761637957616c6c65744e6f74466f756e64002904e85468657265206973206e6f206c65676163792077616c6c657420636f72726573706f6e64696e6720746f20746865207075626c6963206b65792e684c656761637942616c616e63654b65657065724d697373696e67002a081d015468657265206973206e6f206c65676163792062616c616e6365206b65657065722c20736f206e6f206c65676163792077616c6c6574732063616e20626520636c61696d65642e350154686973206973206120636f6e66696775726174696f6e206572726f7220616e642073686f756c64206f6e6c79206f6363757220647572696e67206c6f63616c20646576656c6f706d656e742e3c47756964416c726561647955736564002b04fc5468652073706563696669656420677569642068617320616c7265616479206265656e207573656420616e642063616e6e6f742062652072652d757365642e3c496e76616c69644d61747572697479002c0401015468652076616c7565206f6620746865206c6f616e207465726d2773206d61747572697479206973207a65726f2c20776869636820697320696e76616c69642e04ad010a090909437573746f6d205b6469737061746368206572726f72735d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f7273290a0909096f6620746869732070616c6c65742e0a090909fd0110346672616d655f737570706f72741c73746f726167652c626f756e6465645f76656328426f756e646564566563080454010102045300000400050201185665633c543e00000102084470616c6c65745f646966666963756c747958446966666963756c7479416e6454696d657374616d7004184d6f6d656e74012000080128646966666963756c7479a80128446966666963756c747900012474696d657374616d702001184d6f6d656e740000050200000201020009020c4470616c6c65745f646966666963756c74791870616c6c6574144572726f7204045400010c385a65726f54617267657454696d65000000505a65726f41646a7573746d656e74506572696f64000100604e6567617469766541646a7573746d656e74506572696f6400020004ad010a090909437573746f6d205b6469737061746368206572726f72735d2868747470733a2f2f646f63732e7375627374726174652e696f2f76332f72756e74696d652f6576656e74732d616e642d6572726f7273290a0909096f6620746869732070616c6c65742e0a0909090d02102873705f72756e74696d651c67656e657269634c756e636865636b65645f65787472696e73696348556e636865636b656445787472696e736963101c416464726573730181011043616c6c01a901245369676e617475726501c5011445787472610111020004002800000011020000041c150219021d02210229022d02310200150210306672616d655f73797374656d28657874656e73696f6e7348636865636b5f737065635f76657273696f6e40436865636b5370656356657273696f6e04045400000000190210306672616d655f73797374656d28657874656e73696f6e7340636865636b5f74785f76657273696f6e38436865636b547856657273696f6e040454000000001d0210306672616d655f73797374656d28657874656e73696f6e7334636865636b5f67656e6573697330436865636b47656e6573697304045400000000210210306672616d655f73797374656d28657874656e73696f6e733c636865636b5f6d6f7274616c69747938436865636b4d6f7274616c697479040454000004002502010c45726100002502102873705f72756e74696d651c67656e657269630c6572610c4572610001010420496d6d6f7274616c0000001c4d6f7274616c31040008000001001c4d6f7274616c32040008000002001c4d6f7274616c33040008000003001c4d6f7274616c34040008000004001c4d6f7274616c35040008000005001c4d6f7274616c36040008000006001c4d6f7274616c37040008000007001c4d6f7274616c38040008000008001c4d6f7274616c3904000800000900204d6f7274616c313004000800000a00204d6f7274616c313104000800000b00204d6f7274616c313204000800000c00204d6f7274616c313304000800000d00204d6f7274616c313404000800000e00204d6f7274616c313504000800000f00204d6f7274616c313604000800001000204d6f7274616c313704000800001100204d6f7274616c313804000800001200204d6f7274616c313904000800001300204d6f7274616c323004000800001400204d6f7274616c323104000800001500204d6f7274616c323204000800001600204d6f7274616c323304000800001700204d6f7274616c323404000800001800204d6f7274616c323504000800001900204d6f7274616c323604000800001a00204d6f7274616c323704000800001b00204d6f7274616c323804000800001c00204d6f7274616c323904000800001d00204d6f7274616c333004000800001e00204d6f7274616c333104000800001f00204d6f7274616c333204000800002000204d6f7274616c333304000800002100204d6f7274616c333404000800002200204d6f7274616c333504000800002300204d6f7274616c333604000800002400204d6f7274616c333704000800002500204d6f7274616c333804000800002600204d6f7274616c333904000800002700204d6f7274616c343004000800002800204d6f7274616c343104000800002900204d6f7274616c343204000800002a00204d6f7274616c343304000800002b00204d6f7274616c343404000800002c00204d6f7274616c343504000800002d00204d6f7274616c343604000800002e00204d6f7274616c343704000800002f00204d6f7274616c343804000800003000204d6f7274616c343904000800003100204d6f7274616c353004000800003200204d6f7274616c353104000800003300204d6f7274616c353204000800003400204d6f7274616c353304000800003500204d6f7274616c353404000800003600204d6f7274616c353504000800003700204d6f7274616c353604000800003800204d6f7274616c353704000800003900204d6f7274616c353804000800003a00204d6f7274616c353904000800003b00204d6f7274616c363004000800003c00204d6f7274616c363104000800003d00204d6f7274616c363204000800003e00204d6f7274616c363304000800003f00204d6f7274616c363404000800004000204d6f7274616c363504000800004100204d6f7274616c363604000800004200204d6f7274616c363704000800004300204d6f7274616c363804000800004400204d6f7274616c363904000800004500204d6f7274616c373004000800004600204d6f7274616c373104000800004700204d6f7274616c373204000800004800204d6f7274616c373304000800004900204d6f7274616c373404000800004a00204d6f7274616c373504000800004b00204d6f7274616c373604000800004c00204d6f7274616c373704000800004d00204d6f7274616c373804000800004e00204d6f7274616c373904000800004f00204d6f7274616c383004000800005000204d6f7274616c383104000800005100204d6f7274616c383204000800005200204d6f7274616c383304000800005300204d6f7274616c383404000800005400204d6f7274616c383504000800005500204d6f7274616c383604000800005600204d6f7274616c383704000800005700204d6f7274616c383804000800005800204d6f7274616c383904000800005900204d6f7274616c393004000800005a00204d6f7274616c393104000800005b00204d6f7274616c393204000800005c00204d6f7274616c393304000800005d00204d6f7274616c393404000800005e00204d6f7274616c393504000800005f00204d6f7274616c393604000800006000204d6f7274616c393704000800006100204d6f7274616c393804000800006200204d6f7274616c393904000800006300244d6f7274616c31303004000800006400244d6f7274616c31303104000800006500244d6f7274616c31303204000800006600244d6f7274616c31303304000800006700244d6f7274616c31303404000800006800244d6f7274616c31303504000800006900244d6f7274616c31303604000800006a00244d6f7274616c31303704000800006b00244d6f7274616c31303804000800006c00244d6f7274616c31303904000800006d00244d6f7274616c31313004000800006e00244d6f7274616c31313104000800006f00244d6f7274616c31313204000800007000244d6f7274616c31313304000800007100244d6f7274616c31313404000800007200244d6f7274616c31313504000800007300244d6f7274616c31313604000800007400244d6f7274616c31313704000800007500244d6f7274616c31313804000800007600244d6f7274616c31313904000800007700244d6f7274616c31323004000800007800244d6f7274616c31323104000800007900244d6f7274616c31323204000800007a00244d6f7274616c31323304000800007b00244d6f7274616c31323404000800007c00244d6f7274616c31323504000800007d00244d6f7274616c31323604000800007e00244d6f7274616c31323704000800007f00244d6f7274616c31323804000800008000244d6f7274616c31323904000800008100244d6f7274616c31333004000800008200244d6f7274616c31333104000800008300244d6f7274616c31333204000800008400244d6f7274616c31333304000800008500244d6f7274616c31333404000800008600244d6f7274616c31333504000800008700244d6f7274616c31333604000800008800244d6f7274616c31333704000800008900244d6f7274616c31333804000800008a00244d6f7274616c31333904000800008b00244d6f7274616c31343004000800008c00244d6f7274616c31343104000800008d00244d6f7274616c31343204000800008e00244d6f7274616c31343304000800008f00244d6f7274616c31343404000800009000244d6f7274616c31343504000800009100244d6f7274616c31343604000800009200244d6f7274616c31343704000800009300244d6f7274616c31343804000800009400244d6f7274616c31343904000800009500244d6f7274616c31353004000800009600244d6f7274616c31353104000800009700244d6f7274616c31353204000800009800244d6f7274616c31353304000800009900244d6f7274616c31353404000800009a00244d6f7274616c31353504000800009b00244d6f7274616c31353604000800009c00244d6f7274616c31353704000800009d00244d6f7274616c31353804000800009e00244d6f7274616c31353904000800009f00244d6f7274616c3136300400080000a000244d6f7274616c3136310400080000a100244d6f7274616c3136320400080000a200244d6f7274616c3136330400080000a300244d6f7274616c3136340400080000a400244d6f7274616c3136350400080000a500244d6f7274616c3136360400080000a600244d6f7274616c3136370400080000a700244d6f7274616c3136380400080000a800244d6f7274616c3136390400080000a900244d6f7274616c3137300400080000aa00244d6f7274616c3137310400080000ab00244d6f7274616c3137320400080000ac00244d6f7274616c3137330400080000ad00244d6f7274616c3137340400080000ae00244d6f7274616c3137350400080000af00244d6f7274616c3137360400080000b000244d6f7274616c3137370400080000b100244d6f7274616c3137380400080000b200244d6f7274616c3137390400080000b300244d6f7274616c3138300400080000b400244d6f7274616c3138310400080000b500244d6f7274616c3138320400080000b600244d6f7274616c3138330400080000b700244d6f7274616c3138340400080000b800244d6f7274616c3138350400080000b900244d6f7274616c3138360400080000ba00244d6f7274616c3138370400080000bb00244d6f7274616c3138380400080000bc00244d6f7274616c3138390400080000bd00244d6f7274616c3139300400080000be00244d6f7274616c3139310400080000bf00244d6f7274616c3139320400080000c000244d6f7274616c3139330400080000c100244d6f7274616c3139340400080000c200244d6f7274616c3139350400080000c300244d6f7274616c3139360400080000c400244d6f7274616c3139370400080000c500244d6f7274616c3139380400080000c600244d6f7274616c3139390400080000c700244d6f7274616c3230300400080000c800244d6f7274616c3230310400080000c900244d6f7274616c3230320400080000ca00244d6f7274616c3230330400080000cb00244d6f7274616c3230340400080000cc00244d6f7274616c3230350400080000cd00244d6f7274616c3230360400080000ce00244d6f7274616c3230370400080000cf00244d6f7274616c3230380400080000d000244d6f7274616c3230390400080000d100244d6f7274616c3231300400080000d200244d6f7274616c3231310400080000d300244d6f7274616c3231320400080000d400244d6f7274616c3231330400080000d500244d6f7274616c3231340400080000d600244d6f7274616c3231350400080000d700244d6f7274616c3231360400080000d800244d6f7274616c3231370400080000d900244d6f7274616c3231380400080000da00244d6f7274616c3231390400080000db00244d6f7274616c3232300400080000dc00244d6f7274616c3232310400080000dd00244d6f7274616c3232320400080000de00244d6f7274616c3232330400080000df00244d6f7274616c3232340400080000e000244d6f7274616c3232350400080000e100244d6f7274616c3232360400080000e200244d6f7274616c3232370400080000e300244d6f7274616c3232380400080000e400244d6f7274616c3232390400080000e500244d6f7274616c3233300400080000e600244d6f7274616c3233310400080000e700244d6f7274616c3233320400080000e800244d6f7274616c3233330400080000e900244d6f7274616c3233340400080000ea00244d6f7274616c3233350400080000eb00244d6f7274616c3233360400080000ec00244d6f7274616c3233370400080000ed00244d6f7274616c3233380400080000ee00244d6f7274616c3233390400080000ef00244d6f7274616c3234300400080000f000244d6f7274616c3234310400080000f100244d6f7274616c3234320400080000f200244d6f7274616c3234330400080000f300244d6f7274616c3234340400080000f400244d6f7274616c3234350400080000f500244d6f7274616c3234360400080000f600244d6f7274616c3234370400080000f700244d6f7274616c3234380400080000f800244d6f7274616c3234390400080000f900244d6f7274616c3235300400080000fa00244d6f7274616c3235310400080000fb00244d6f7274616c3235320400080000fc00244d6f7274616c3235330400080000fd00244d6f7274616c3235340400080000fe00244d6f7274616c3235350400080000ff0000290210306672616d655f73797374656d28657874656e73696f6e732c636865636b5f6e6f6e636528436865636b4e6f6e63650404540000040001010120543a3a496e64657800002d0210306672616d655f73797374656d28657874656e73696f6e7330636865636b5f7765696768742c436865636b576569676874040454000000003102086870616c6c65745f7472616e73616374696f6e5f7061796d656e74604368617267655472616e73616374696f6e5061796d656e74040454000004008d01013042616c616e63654f663c543e00003502085c637265646974636f696e5f6e6f64655f72756e74696d651c52756e74696d6500000000201853797374656d011853797374656d401c4163636f756e7401010402000c4101000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004e8205468652066756c6c206163636f756e7420696e666f726d6174696f6e20666f72206120706172746963756c6172206163636f756e742049442e3845787472696e736963436f756e74000010040004b820546f74616c2065787472696e7369637320636f756e7420666f72207468652063757272656e7420626c6f636b2e2c426c6f636b57656967687401001c600000000000000000000000000000000000000000000000000488205468652063757272656e742077656967687420666f722074686520626c6f636b2e40416c6c45787472696e736963734c656e000010040004410120546f74616c206c656e6774682028696e2062797465732920666f7220616c6c2065787472696e736963732070757420746f6765746865722c20666f72207468652063757272656e7420626c6f636b2e24426c6f636b486173680101040510248000000000000000000000000000000000000000000000000000000000000000000498204d6170206f6620626c6f636b206e756d6265727320746f20626c6f636b206861736865732e3445787472696e736963446174610101040510280400043d012045787472696e73696373206461746120666f72207468652063757272656e7420626c6f636b20286d61707320616e2065787472696e736963277320696e64657820746f206974732064617461292e184e756d6265720100101000000000040901205468652063757272656e7420626c6f636b206e756d626572206265696e672070726f6365737365642e205365742062792060657865637574655f626c6f636b602e28506172656e744861736801002480000000000000000000000000000000000000000000000000000000000000000004702048617368206f66207468652070726576696f757320626c6f636b2e1844696765737401002c040004f020446967657374206f66207468652063757272656e7420626c6f636b2c20616c736f2070617274206f662074686520626c6f636b206865616465722e184576656e747301003c040010a0204576656e7473206465706f736974656420666f72207468652063757272656e7420626c6f636b2e005901204e4f54453a20546869732073746f72616765206974656d206973206578706c696369746c7920756e626f756e6465642073696e6365206974206973206e6576657220696e74656e64656420746f2062652072656164642066726f6d2077697468696e207468652072756e74696d652e284576656e74436f756e74010010100000000004b820546865206e756d626572206f66206576656e747320696e2074686520604576656e74733c543e60206c6973742e2c4576656e74546f706963730101040224f40400282501204d617070696e67206265747765656e206120746f7069632028726570726573656e74656420627920543a3a486173682920616e64206120766563746f72206f6620696e646578657394206f66206576656e747320696e2074686520603c4576656e74733c543e3e60206c6973742e00510120416c6c20746f70696320766563746f727320686176652064657465726d696e69737469632073746f72616765206c6f636174696f6e7320646570656e64696e67206f6e2074686520746f7069632e2054686973450120616c6c6f7773206c696768742d636c69656e747320746f206c6576657261676520746865206368616e67657320747269652073746f7261676520747261636b696e67206d656368616e69736d20616e64e420696e2063617365206f66206368616e67657320666574636820746865206c697374206f66206576656e7473206f6620696e7465726573742e004d01205468652076616c756520686173207468652074797065206028543a3a426c6f636b4e756d6265722c204576656e74496e646578296020626563617573652069662077652075736564206f6e6c79206a7573744d012074686520604576656e74496e64657860207468656e20696e20636173652069662074686520746f70696320686173207468652073616d6520636f6e74656e7473206f6e20746865206e65787420626c6f636b0101206e6f206e6f74696669636174696f6e2077696c6c20626520747269676765726564207468757320746865206576656e74206d69676874206265206c6f73742e484c61737452756e74696d65557067726164650000fc04000455012053746f726573207468652060737065635f76657273696f6e6020616e642060737065635f6e616d6560206f66207768656e20746865206c6173742072756e74696d6520757067726164652068617070656e65642e545570677261646564546f553332526566436f756e740100b00400044d012054727565206966207765206861766520757067726164656420736f207468617420607479706520526566436f756e74602069732060753332602e2046616c7365202864656661756c7429206966206e6f742e605570677261646564546f547269706c65526566436f756e740100b00400085d012054727565206966207765206861766520757067726164656420736f2074686174204163636f756e74496e666f20636f6e7461696e73207468726565207479706573206f662060526566436f756e74602e2046616c736548202864656661756c7429206966206e6f742e38457865637574696f6e50686173650000ec040004882054686520657865637574696f6e207068617365206f662074686520626c6f636b2e01090101481830426c6f636b576569676874731d01850100f2052a010000000040e59c30120000405973070000000001c076eec4d20b00000100f0ab75a40d0000010000000000000000405973070000000001c0c627ec5e100000010040e59c3012000001005039278c040000405973070000000000000004d020426c6f636b20262065787472696e7369637320776569676874733a20626173652076616c75657320616e64206c696d6974732e2c426c6f636b4c656e6774682d013000003c00000050000000500004a820546865206d6178696d756d206c656e677468206f66206120626c6f636b2028696e206279746573292e38426c6f636b48617368436f756e74101060090000045501204d6178696d756d206e756d626572206f6620626c6f636b206e756d62657220746f20626c6f636b2068617368206d617070696e677320746f206b65657020286f6c64657374207072756e6564206669727374292e20446257656967687435014040787d010000000000e1f505000000000409012054686520776569676874206f662072756e74696d65206461746162617365206f7065726174696f6e73207468652072756e74696d652063616e20696e766f6b652e1c56657273696f6e3901a9023c637265646974636f696e2d6e6f64653c637265646974636f696e2d6e6f646501000000c80000000100000028df6acb689907609b0400000037e397fc7c91f5e40100000040fe3ad401f8959a05000000d2bc9897eed08f1503000000f78b278be53f454c02000000ab3c0572291feb8b01000000bc9d89904f5b923f0100000037c8bb1350a9a2a8010000001ea173a1db199b3b010000006c7049b21e2444110100000002000000010484204765742074686520636861696e27732063757272656e742076657273696f6e2e28535335385072656669784d01082a0014a8205468652064657369676e61746564205353383520707265666978206f66207468697320636861696e2e0039012054686973207265706c6163657320746865202273733538466f726d6174222070726f7065727479206465636c6172656420696e2074686520636861696e20737065632e20526561736f6e20697331012074686174207468652072756e74696d652073686f756c64206b6e6f772061626f7574207468652070726566697820696e206f7264657220746f206d616b6520757365206f662069742061737020616e206964656e746966696572206f662074686520636861696e2e015101002454696d657374616d70012454696d657374616d70080c4e6f7701002020000000000000000004902043757272656e742074696d6520666f72207468652063757272656e7420626c6f636b2e244469645570646174650100b0040004b420446964207468652074696d657374616d7020676574207570646174656420696e207468697320626c6f636b3f0155010004344d696e696d756d506572696f642020e803000000000000104d0120546865206d696e696d756d20706572696f64206265747765656e20626c6f636b732e204265776172652074686174207468697320697320646966666572656e7420746f20746865202a65787065637465642a5d0120706572696f6420746861742074686520626c6f636b2070726f64756374696f6e206170706172617475732070726f76696465732e20596f75722063686f73656e20636f6e73656e7375732073797374656d2077696c6c5d012067656e6572616c6c7920776f726b2077697468207468697320746f2064657465726d696e6520612073656e7369626c6520626c6f636b2074696d652e20652e672e20466f7220417572612c2069742077696c6c206265a020646f75626c65207468697320706572696f64206f6e2064656661756c742073657474696e67732e00012042616c616e636573012042616c616e6365731434546f74616c49737375616e6365010018400000000000000000000000000000000004982054686520746f74616c20756e6974732069737375656420696e207468652073797374656d2e1c4163636f756e740101040200140101000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c6c205468652062616c616e6365206f6620616e206163636f756e742e004101204e4f54453a2054686973206973206f6e6c79207573656420696e207468652063617365207468617420746869732070616c6c6574206973207573656420746f2073746f72652062616c616e6365732e144c6f636b7301010402005d01040008b820416e79206c6971756964697479206c6f636b73206f6e20736f6d65206163636f756e742062616c616e6365732e2501204e4f54453a2053686f756c64206f6e6c79206265206163636573736564207768656e2073657474696e672c206368616e67696e6720616e642066726565696e672061206c6f636b2e20526573657276657301010402006d01040004a4204e616d6564207265736572766573206f6e20736f6d65206163636f756e742062616c616e6365732e3853746f7261676556657273696f6e0100790104000c7c2053746f726167652076657273696f6e206f66207468652070616c6c65742e00a020546869732069732073657420746f2076322e302e3020666f72206e6577206e6574776f726b732e017d0101640c484578697374656e7469616c4465706f7369741840f401000000000000000000000000000004d420546865206d696e696d756d20616d6f756e7420726571756972656420746f206b65657020616e206163636f756e74206f70656e2e204d61784c6f636b7310103200000008f420546865206d6178696d756d206e756d626572206f66206c6f636b7320746861742073686f756c64206578697374206f6e20616e206163636f756e742edc204e6f74207374726963746c7920656e666f726365642c20627574207573656420666f722077656967687420657374696d6174696f6e2e2c4d61785265736572766573101000000000040d0120546865206d6178696d756d206e756d626572206f66206e616d656420726573657276657320746861742063616e206578697374206f6e20616e206163636f756e742e01910102485472616e73616374696f6e5061796d656e7401485472616e73616374696f6e5061796d656e7408444e6578744665654d756c7469706c6965720100950140000064a7b3b6e00d0000000000000000003853746f7261676556657273696f6e0100990104000000000c485472616e73616374696f6e42797465466565184001000000000000000000000000000000040d01205468652066656520746f206265207061696420666f72206d616b696e672061207472616e73616374696f6e3b20746865207065722d6279746520706f7274696f6e2e604f7065726174696f6e616c4665654d756c7469706c696572080401545901204120666565206d756c6974706c69657220666f7220604f7065726174696f6e616c602065787472696e7369637320746f20636f6d7075746520227669727475616c207469702220746f20626f6f73742074686569722c20607072696f7269747960004d0120546869732076616c7565206973206d756c7469706c656420627920746865206066696e616c5f6665656020746f206f627461696e206120227669727475616c20746970222074686174206973206c61746572f420616464656420746f20612074697020636f6d706f6e656e7420696e20726567756c617220607072696f72697479602063616c63756c6174696f6e732e4d01204974206d65616e732074686174206120604e6f726d616c60207472616e73616374696f6e2063616e2066726f6e742d72756e20612073696d696c61726c792d73697a656420604f7065726174696f6e616c6041012065787472696e736963202877697468206e6f20746970292c20627920696e636c7564696e672061207469702076616c75652067726561746572207468616e20746865207669727475616c207469702e003c20606060727573742c69676e6f726540202f2f20466f7220604e6f726d616c608c206c6574207072696f72697479203d207072696f726974795f63616c6328746970293b0054202f2f20466f7220604f7065726174696f6e616c601101206c6574207669727475616c5f746970203d2028696e636c7573696f6e5f666565202b2074697029202a204f7065726174696f6e616c4665654d756c7469706c6965723bc4206c6574207072696f72697479203d207072696f726974795f63616c6328746970202b207669727475616c5f746970293b1020606060005101204e6f746520746861742073696e636520776520757365206066696e616c5f6665656020746865206d756c7469706c696572206170706c69657320616c736f20746f2074686520726567756c61722060746970605d012073656e74207769746820746865207472616e73616374696f6e2e20536f2c206e6f74206f6e6c7920646f657320746865207472616e73616374696f6e206765742061207072696f726974792062756d702062617365646101206f6e207468652060696e636c7573696f6e5f666565602c2062757420776520616c736f20616d706c6966792074686520696d70616374206f662074697073206170706c69656420746f20604f7065726174696f6e616c6038207472616e73616374696f6e732e2c576569676874546f4665659d015c04005a6202000000000000000000000000000000000001040d012054686520706f6c796e6f6d69616c2074686174206973206170706c69656420696e206f7264657220746f20646572697665206665652066726f6d207765696768742e0003105375646f01105375646f040c4b6579000000040004842054686520604163636f756e74496460206f6620746865207375646f206b65792e01a501016c0001e5010428437265646974636f696e0128437265646974636f696e2c2c417574686f726974696573000104020074040000344c656761637957616c6c65747300010403e0180400004c4c656761637942616c616e63654b65657065720000000400004c556e76657269666965645472616e73666572730100e901040000284465616c4f72646572730001080506f501d80400002441646472657373657300010402808404000024557365644775696473000104028c740400002441736b4f72646572730001080506f501b8040000244269644f72646572730001080506f501c8040000184f66666572730001080506f501d4040000245472616e736665727300010406909404000001ad01017c0001f9010528446966666963756c74790128446966666963756c7479108450726576696f7573446966666963756c74696573416e6454696d657374616d70730100fd010400004443757272656e74446966666963756c74790100a8800000000000000000000000000000000000000000000000000000000000000000003c546172676574426c6f636b54696d650100202000000000000000000068446966666963756c747941646a7573746d656e74506572696f640100e1012000000000000000000001dd010000010902061c52657761726473011c52657761726473042c426c6f636b417574686f720000000400000001e80000070d02041c40436865636b5370656356657273696f6e15021038436865636b547856657273696f6e19021030436865636b47656e657369731d022438436865636b4d6f7274616c69747921022428436865636b4e6f6e63652902742c436865636b5765696768742d0274604368617267655472616e73616374696f6e5061796d656e743102743502","id":"1"}
diff --git a/java-script/src/interfaces/augment-api-errors.ts b/java-script/src/interfaces/augment-api-errors.ts
index 7479ae5..baa92a7 100644
--- a/java-script/src/interfaces/augment-api-errors.ts
+++ b/java-script/src/interfaces/augment-api-errors.ts
@@ -48,54 +48,186 @@ declare module '@polkadot/api-base/types/errors' {
        * The specified address has already been registered to another account
        **/
       AddressAlreadyRegistered: AugmentedError<ApiType>;
+      /**
+       * The addresses specified are not on compatible external chains.
+       **/
       AddressPlatformMismatch: AugmentedError<ApiType>;
+      /**
+       * The account is already an authority.
+       **/
       AlreadyAuthority: AugmentedError<ApiType>;
+      /**
+       * The terms of the ask and bid order do not agree.
+       **/
       AskBidMismatch: AugmentedError<ApiType>;
+      /**
+       * The ask order has expired and is no longer valid.
+       **/
       AskOrderExpired: AugmentedError<ApiType>;
+      /**
+       * The bid order has expired and is no longer valid.
+       **/
       BidOrderExpired: AugmentedError<ApiType>;
+      /**
+       * The deal cannot be locked because it is not funded yet.
+       **/
       DealNotFunded: AugmentedError<ApiType>;
+      /**
+       * The deal order is already closed and cannot be closed again.
+       **/
       DealOrderAlreadyClosed: AugmentedError<ApiType>;
+      /**
+       * The deal order is already funded and cannot be funded again.
+       **/
       DealOrderAlreadyFunded: AugmentedError<ApiType>;
+      /**
+       * The deal order is already locked and cannot be locked again.
+       **/
       DealOrderAlreadyLocked: AugmentedError<ApiType>;
+      /**
+       * The deal order has expired and is no longer valid.
+       **/
       DealOrderExpired: AugmentedError<ApiType>;
+      /**
+       * The deal order must be locked before it can be closed.
+       **/
       DealOrderMustBeLocked: AugmentedError<ApiType>;
+      /**
+       * The deal order already exists.
+       **/
       DuplicateDealOrder: AugmentedError<ApiType>;
+      /**
+       * The specified ID has already been used.
+       **/
       DuplicateId: AugmentedError<ApiType>;
+      /**
+       * The offer has already been made.
+       **/
       DuplicateOffer: AugmentedError<ApiType>;
+      /**
+       * The specified guid has already been used and cannot be re-used.
+       **/
       GuidAlreadyUsed: AugmentedError<ApiType>;
+      /**
+       * The node does not have sufficient authority to verify a transfer.
+       **/
       InsufficientAuthority: AugmentedError<ApiType>;
+      /**
+       * The value of the loan term's maturity is zero, which is invalid.
+       **/
       InvalidMaturity: AugmentedError<ApiType>;
+      /**
+       * The signature does not match the public key and message.
+       **/
       InvalidSignature: AugmentedError<ApiType>;
+      /**
+       * There is no legacy balance keeper, so no legacy wallets can be claimed.
+       * This is a configuration error and should only occur during local development.
+       **/
       LegacyBalanceKeeperMissing: AugmentedError<ApiType>;
-      LegacySighashMalformed: AugmentedError<ApiType>;
+      /**
+       * There is no legacy wallet corresponding to the public key.
+       **/
       LegacyWalletNotFound: AugmentedError<ApiType>;
+      /**
+       * The deal order is malformed and has a block number greater than the
+       * tip. This is an internal error.
+       **/
       MalformedDealOrder: AugmentedError<ApiType>;
+      /**
+       * The transfer is malformed and has a block number greater than the
+       * tip. This is an internal error.
+       **/
       MalformedTransfer: AugmentedError<ApiType>;
+      /**
+       * The node is an authority but there is no account to create a
+       * callback transaction. This is likely an internal error.
+       **/
       NoLocalAcctForSignedTx: AugmentedError<ApiType>;
+      /**
+       * The specified address does not exist.
+       **/
       NonExistentAddress: AugmentedError<ApiType>;
+      /**
+       * The specified ask order does not exist.
+       **/
       NonExistentAskOrder: AugmentedError<ApiType>;
+      /**
+       * The specified bid order does not exist.
+       **/
       NonExistentBidOrder: AugmentedError<ApiType>;
+      /**
+       * The specified deal order does not exist.
+       **/
       NonExistentDealOrder: AugmentedError<ApiType>;
+      /**
+       * The specified offer does not exist.
+       **/
       NonExistentOffer: AugmentedError<ApiType>;
-      NonExistentRepaymentOrder: AugmentedError<ApiType>;
+      /**
+       * The specified transfer does not exist.
+       **/
       NonExistentTransfer: AugmentedError<ApiType>;
+      /**
+       * The address cannot be used because the user does not own it.
+       **/
       NotAddressOwner: AugmentedError<ApiType>;
+      /**
+       * Only the borrower can perform the action.
+       **/
       NotBorrower: AugmentedError<ApiType>;
+      /**
+       * The legacy wallet is not owned by the user.
+       **/
       NotLegacyWalletOwner: AugmentedError<ApiType>;
+      /**
+       * Only the lender can perform the action.
+       **/
       NotLender: AugmentedError<ApiType>;
+      /**
+       * Failed to send an offchain callback transaction. This is likely
+       * an internal error.
+       **/
       OffchainSignedTxFailed: AugmentedError<ApiType>;
+      /**
+       * The offer order has expired and is no longer valid.
+       **/
       OfferExpired: AugmentedError<ApiType>;
       RepaymentOrderNonZeroGain: AugmentedError<ApiType>;
+      /**
+       * Repayment orders are not currently supported.
+       **/
       RepaymentOrderUnsupported: AugmentedError<ApiType>;
+      /**
+       * The bid order is owned by the user, a user cannot lend to themself.
+       **/
       SameOwner: AugmentedError<ApiType>;
-      ScaleDecodeError: AugmentedError<ApiType>;
+      /**
+       * The transfer has already been processed and cannot be used.
+       **/
       TransferAlreadyProcessed: AugmentedError<ApiType>;
+      /**
+       * The transfer has already been registered.
+       **/
       TransferAlreadyRegistered: AugmentedError<ApiType>;
+      /**
+       * The transfer amount is less than the amount in the loan terms.
+       **/
       TransferAmountInsufficient: AugmentedError<ApiType>;
+      /**
+       * The account that registered the transfer does
+       * not match the account attempting to use the transfer.
+       **/
       TransferMismatch: AugmentedError<ApiType>;
+      /**
+       * The specified transfer type is not currently supported by
+       * the blockchain the loan is executed on.
+       **/
       UnsupportedTransferKind: AugmentedError<ApiType>;
+      /**
+       * The queue of unverified transfers is full for this block.
+       **/
       UnverifiedTransferPoolFull: AugmentedError<ApiType>;
-      VerifyStringTooLong: AugmentedError<ApiType>;
       /**
        * Generic error
        **/
diff --git a/java-script/src/interfaces/augment-api-events.ts b/java-script/src/interfaces/augment-api-events.ts
index 1159390..2e0de85 100644
--- a/java-script/src/interfaces/augment-api-events.ts
+++ b/java-script/src/interfaces/augment-api-events.ts
@@ -58,20 +58,74 @@ declare module '@polkadot/api-base/types/events' {
     };
     creditcoin: {
       /**
-       * Event documentation should end with an array that provides descriptive names for event
-       * parameters. [something, who]
+       * An address on an external chain has been registered.
+       * [registered_address_id, registered_address]
        **/
       AddressRegistered: AugmentedEvent<ApiType, [H256, PalletCreditcoinAddress]>;
+      /**
+       * An ask order has been added by a prospective lender. This indicates that the lender
+       * is looking to issue a loan with certain terms.
+       * [ask_order_id, ask_order]
+       **/
       AskOrderAdded: AugmentedEvent<ApiType, [PalletCreditcoinAskOrderId, PalletCreditcoinAskOrder]>;
+      /**
+       * A bid order has been added by a prospective borrower. This indicates that the borrower
+       * is looking for a loan with certain terms.
+       * [bid_order_id, bid_order]
+       **/
       BidOrderAdded: AugmentedEvent<ApiType, [PalletCreditcoinBidOrderId, PalletCreditcoinBidOrder]>;
+      /**
+       * A deal order has been added by a borrower. This indicates that the borrower
+       * has accepted a lender's offer and intends to enter the loan.
+       * [deal_order_id, deal_order]
+       **/
       DealOrderAdded: AugmentedEvent<ApiType, [PalletCreditcoinDealOrderId, PalletCreditcoinDealOrder]>;
+      /**
+       * A deal order has been closed by a borrower. This indicates that the borrower
+       * has repaid the loan in full and is now closing out the loan.
+       * [closed_deal_order_id, closed_deal_order]
+       **/
       DealOrderClosed: AugmentedEvent<ApiType, [PalletCreditcoinDealOrderId, PalletCreditcoinDealOrder]>;
+      /**
+       * A deal order has been funded by a lender. This indicates that the lender
+       * has initiated the actual loan by transferring the loan amount to the borrower
+       * on an external chain.
+       * [funded_deal_order_id, funded_deal_order]
+       **/
       DealOrderFunded: AugmentedEvent<ApiType, [PalletCreditcoinDealOrderId, PalletCreditcoinDealOrder]>;
+      /**
+       * A legacy wallet from Creditcoin 1.X has been claimed. The balance of the legacy wallet
+       * has been transferred to the owner's Creditcoin 2.0 account.
+       * [legacy_wallet_claimer, legacy_wallet_sighash, legacy_wallet_balance]
+       **/
       LegacyWalletClaimed: AugmentedEvent<ApiType, [AccountId32, PalletCreditcoinLegacySighash, u128]>;
+      /**
+       * A loan exemption has been granted by a lender. This indicates that the lender
+       * is releasing some or all of the outstanding debt on the loan. The borrower
+       * is no longer responsible for repaying the amount.
+       * [exempted_deal_order_id, exempting_transfer_id]
+       **/
       LoanExempted: AugmentedEvent<ApiType, [PalletCreditcoinDealOrderId, H256]>;
+      /**
+       * An offer has been added by a lender. This indicates that the lender
+       * is interested in entering a loan with the owner of the bid order.
+       * [offer_id, offer]
+       **/
       OfferAdded: AugmentedEvent<ApiType, [PalletCreditcoinOfferId, PalletCreditcoinOffer]>;
+      /**
+       * An external transfer has been processed and marked as part of a loan.
+       * [processed_transfer_id, processed_transfer]
+       **/
       TransferProcessed: AugmentedEvent<ApiType, [H256, PalletCreditcoinTransfer]>;
+      /**
+       * An external transfer has been registered and will be verified.
+       * [registered_transfer_id, registered_transfer]
+       **/
       TransferRegistered: AugmentedEvent<ApiType, [H256, PalletCreditcoinTransfer]>;
+      /**
+       * An external transfer has been successfully verified.
+       * [verified_transfer_id, verified_transfer]
+       **/
       TransferVerified: AugmentedEvent<ApiType, [H256, PalletCreditcoinTransfer]>;
       /**
        * Generic event
diff --git a/java-script/src/interfaces/augment-api-rpc.ts b/java-script/src/interfaces/augment-api-rpc.ts
index 37edc06..1c0e5a0 100644
--- a/java-script/src/interfaces/augment-api-rpc.ts
+++ b/java-script/src/interfaces/augment-api-rpc.ts
@@ -12,6 +12,7 @@ import type { BlockHash } from '@polkadot/types/interfaces/chain';
 import type { PrefixedStorageKey } from '@polkadot/types/interfaces/childstate';
 import type { AuthorityId } from '@polkadot/types/interfaces/consensus';
 import type { CodeUploadRequest, CodeUploadResult, ContractCallRequest, ContractExecResult, ContractInstantiateResult, InstantiateRequest } from '@polkadot/types/interfaces/contracts';
+import type { BlockStats } from '@polkadot/types/interfaces/dev';
 import type { CreatedBlock } from '@polkadot/types/interfaces/engine';
 import type { EthAccount, EthCallRequest, EthFilter, EthFilterChanges, EthLog, EthReceipt, EthRichBlock, EthSubKind, EthSubParams, EthSyncStatus, EthTransaction, EthTransactionRequest, EthWork } from '@polkadot/types/interfaces/eth';
 import type { Extrinsic } from '@polkadot/types/interfaces/extrinsics';
@@ -21,7 +22,7 @@ import type { StorageKind } from '@polkadot/types/interfaces/offchain';
 import type { FeeDetails, RuntimeDispatchInfo } from '@polkadot/types/interfaces/payment';
 import type { RpcMethods } from '@polkadot/types/interfaces/rpc';
 import type { AccountId, BlockNumber, H160, H256, H64, Hash, Header, Index, Justification, KeyValue, SignedBlock, StorageData } from '@polkadot/types/interfaces/runtime';
-import type { ReadProof, RuntimeVersion, TraceBlockResponse } from '@polkadot/types/interfaces/state';
+import type { MigrationStatusResult, ReadProof, RuntimeVersion, TraceBlockResponse } from '@polkadot/types/interfaces/state';
 import type { ApplyExtrinsicResult, ChainProperties, ChainType, Health, NetworkState, NodeRole, PeerInfo, SyncState } from '@polkadot/types/interfaces/system';
 import type { IExtrinsic, Observable } from '@polkadot/types/types';
 
@@ -155,6 +156,12 @@ declare module '@polkadot/rpc-core/types/jsonrpc' {
        **/
       uploadCode: AugmentedRpc<(uploadRequest: CodeUploadRequest | { origin?: any; code?: any; storageDepositLimit?: any } | string | Uint8Array, at?: BlockHash | string | Uint8Array) => Observable<CodeUploadResult>>;
     };
+    dev: {
+      /**
+       * Reexecute the specified `block_hash` and gather statistics while doing so
+       **/
+      getBlockStats: AugmentedRpc<(at: Hash | string | Uint8Array) => Observable<Option<BlockStats>>>;
+    };
     engine: {
       /**
        * Instructs the manual-seal authorship task to create a new block
@@ -472,6 +479,10 @@ declare module '@polkadot/rpc-core/types/jsonrpc' {
        * Provides a way to trace the re-execution of a single block
        **/
       traceBlock: AugmentedRpc<(block: Hash | string | Uint8Array, targets: Option<Text> | null | object | string | Uint8Array, storageKeys: Option<Text> | null | object | string | Uint8Array, methods: Option<Text> | null | object | string | Uint8Array) => Observable<TraceBlockResponse>>;
+      /**
+       * Check current migration state
+       **/
+      trieMigrationStatus: AugmentedRpc<(at?: BlockHash | string | Uint8Array) => Observable<MigrationStatusResult>>;
     };
     syncstate: {
       /**
diff --git a/java-script/src/interfaces/augment-types.ts b/java-script/src/interfaces/augment-types.ts
index a767ecd..9a334b8 100644
--- a/java-script/src/interfaces/augment-types.ts
+++ b/java-script/src/interfaces/augment-types.ts
@@ -22,6 +22,7 @@ import type { ContractConstructorSpecLatest, ContractConstructorSpecV0, Contract
 import type { FundIndex, FundInfo, LastContribution, TrieIndex } from '@polkadot/types/interfaces/crowdloan';
 import type { ConfigData, MessageId, OverweightIndex, PageCounter, PageIndexData } from '@polkadot/types/interfaces/cumulus';
 import type { AccountVote, AccountVoteSplit, AccountVoteStandard, Conviction, Delegations, PreimageStatus, PreimageStatusAvailable, PriorLock, PropIndex, Proposal, ProxyState, ReferendumIndex, ReferendumInfo, ReferendumInfoFinished, ReferendumInfoTo239, ReferendumStatus, Tally, Voting, VotingDelegating, VotingDirect, VotingDirectVote } from '@polkadot/types/interfaces/democracy';
+import type { BlockStats } from '@polkadot/types/interfaces/dev';
 import type { ApprovalFlag, DefunctVoter, Renouncing, SetIndex, Vote, VoteIndex, VoteThreshold, VoterInfo } from '@polkadot/types/interfaces/elections';
 import type { CreatedBlock, ImportedAux } from '@polkadot/types/interfaces/engine';
 import type { BlockV0, BlockV1, BlockV2, EIP1559Transaction, EIP2930Transaction, EthAccessList, EthAccessListItem, EthAccount, EthAddress, EthBlock, EthBloom, EthCallRequest, EthFilter, EthFilterAddress, EthFilterChanges, EthFilterTopic, EthFilterTopicEntry, EthFilterTopicInner, EthHeader, EthLog, EthReceipt, EthRichBlock, EthRichHeader, EthStorageProof, EthSubKind, EthSubParams, EthSubResult, EthSyncInfo, EthSyncStatus, EthTransaction, EthTransactionAction, EthTransactionCondition, EthTransactionRequest, EthTransactionSignature, EthTransactionStatus, EthWork, EthereumAccountId, EthereumAddress, EthereumLookupSource, EthereumSignature, LegacyTransaction, TransactionV0, TransactionV1, TransactionV2 } from '@polkadot/types/interfaces/eth';
@@ -50,7 +51,7 @@ import type { Period, Priority, SchedulePeriod, SchedulePriority, Scheduled, Sch
 import type { BeefyKey, FullIdentification, IdentificationTuple, Keys, MembershipProof, SessionIndex, SessionKeys1, SessionKeys10, SessionKeys10B, SessionKeys2, SessionKeys3, SessionKeys4, SessionKeys5, SessionKeys6, SessionKeys6B, SessionKeys7, SessionKeys7B, SessionKeys8, SessionKeys8B, SessionKeys9, SessionKeys9B, ValidatorCount } from '@polkadot/types/interfaces/session';
 import type { Bid, BidKind, SocietyJudgement, SocietyVote, StrikeCount, VouchingStatus } from '@polkadot/types/interfaces/society';
 import type { ActiveEraInfo, CompactAssignments, CompactAssignmentsTo257, CompactAssignmentsTo265, CompactAssignmentsWith16, CompactAssignmentsWith24, CompactScore, CompactScoreCompact, ElectionCompute, ElectionPhase, ElectionResult, ElectionScore, ElectionSize, ElectionStatus, EraIndex, EraPoints, EraRewardPoints, EraRewards, Exposure, ExtendedBalance, Forcing, IndividualExposure, KeyType, MomentOf, Nominations, NominatorIndex, NominatorIndexCompact, OffchainAccuracy, OffchainAccuracyCompact, PhragmenScore, Points, RawSolution, RawSolutionTo265, RawSolutionWith16, RawSolutionWith24, ReadySolution, RewardDestination, RewardPoint, RoundSnapshot, SeatHolder, SignedSubmission, SignedSubmissionOf, SignedSubmissionTo276, SlashJournalEntry, SlashingSpans, SlashingSpansTo204, SolutionOrSnapshotSize, SolutionSupport, SolutionSupports, SpanIndex, SpanRecord, StakingLedger, StakingLedgerTo223, StakingLedgerTo240, SubmissionIndicesOf, Supports, UnappliedSlash, UnappliedSlashOther, UnlockChunk, ValidatorIndex, ValidatorIndexCompact, ValidatorPrefs, ValidatorPrefsTo145, ValidatorPrefsTo196, ValidatorPrefsWithBlocked, ValidatorPrefsWithCommission, VoteWeight, Voter } from '@polkadot/types/interfaces/staking';
-import type { ApiId, BlockTrace, BlockTraceEvent, BlockTraceEventData, BlockTraceSpan, KeyValueOption, ReadProof, RuntimeVersion, RuntimeVersionApi, RuntimeVersionPartial, SpecVersion, StorageChangeSet, TraceBlockResponse, TraceError } from '@polkadot/types/interfaces/state';
+import type { ApiId, BlockTrace, BlockTraceEvent, BlockTraceEventData, BlockTraceSpan, KeyValueOption, MigrationStatusResult, ReadProof, RuntimeVersion, RuntimeVersionApi, RuntimeVersionPartial, SpecVersion, StorageChangeSet, TraceBlockResponse, TraceError } from '@polkadot/types/interfaces/state';
 import type { WeightToFeeCoefficient } from '@polkadot/types/interfaces/support';
 import type { AccountInfo, AccountInfoWithDualRefCount, AccountInfoWithProviders, AccountInfoWithRefCount, AccountInfoWithRefCountU8, AccountInfoWithTripleRefCount, ApplyExtrinsicResult, ArithmeticError, BlockLength, BlockWeights, ChainProperties, ChainType, ConsumedWeight, DigestOf, DispatchClass, DispatchError, DispatchErrorModule, DispatchErrorTo198, DispatchInfo, DispatchInfoTo190, DispatchInfoTo244, DispatchOutcome, DispatchResult, DispatchResultOf, DispatchResultTo198, Event, EventId, EventIndex, EventRecord, Health, InvalidTransaction, Key, LastRuntimeUpgradeInfo, NetworkState, NetworkStatePeerset, NetworkStatePeersetInfo, NodeRole, NotConnectedPeer, Peer, PeerEndpoint, PeerEndpointAddr, PeerInfo, PeerPing, PerDispatchClassU32, PerDispatchClassWeight, PerDispatchClassWeightsPerClass, Phase, RawOrigin, RefCount, RefCountTo259, SyncState, SystemOrigin, TokenError, TransactionValidityError, UnknownTransaction, WeightPerClass } from '@polkadot/types/interfaces/system';
 import type { Bounty, BountyIndex, BountyStatus, BountyStatusActive, BountyStatusCuratorProposed, BountyStatusPendingPayout, OpenTip, OpenTipFinderTo225, OpenTipTip, OpenTipTo225, TreasuryProposal } from '@polkadot/types/interfaces/treasury';
@@ -152,6 +153,7 @@ declare module '@polkadot/types/types/registry' {
     BlockNumber: BlockNumber;
     BlockNumberFor: BlockNumberFor;
     BlockNumberOf: BlockNumberOf;
+    BlockStats: BlockStats;
     BlockTrace: BlockTrace;
     BlockTraceEvent: BlockTraceEvent;
     BlockTraceEventData: BlockTraceEventData;
@@ -591,6 +593,7 @@ declare module '@polkadot/types/types/registry' {
     MetadataV13: MetadataV13;
     MetadataV14: MetadataV14;
     MetadataV9: MetadataV9;
+    MigrationStatusResult: MigrationStatusResult;
     MmrLeafProof: MmrLeafProof;
     MmrRootHash: MmrRootHash;
     ModuleConstantMetadataV10: ModuleConstantMetadataV10;
diff --git a/java-script/src/interfaces/lookup.ts b/java-script/src/interfaces/lookup.ts
index 0214ae8..b5ad180 100644
--- a/java-script/src/interfaces/lookup.ts
+++ b/java-script/src/interfaces/lookup.ts
@@ -766,7 +766,7 @@ export default {
    * Lookup126: pallet_creditcoin::pallet::Error<T>
    **/
   PalletCreditcoinError: {
-    _enum: ['AddressAlreadyRegistered', 'NonExistentAddress', 'NonExistentDealOrder', 'NonExistentAskOrder', 'NonExistentBidOrder', 'NonExistentOffer', 'NonExistentTransfer', 'TransferAlreadyRegistered', 'TransferMismatch', 'TransferAlreadyProcessed', 'TransferAmountInsufficient', 'MalformedTransfer', 'UnsupportedTransferKind', 'InsufficientAuthority', 'NonExistentRepaymentOrder', 'DuplicateId', 'NotAddressOwner', 'OffchainSignedTxFailed', 'NoLocalAcctForSignedTx', 'RepaymentOrderNonZeroGain', 'AddressPlatformMismatch', 'AlreadyAuthority', 'DuplicateOffer', 'DealNotFunded', 'DealOrderAlreadyFunded', 'DealOrderAlreadyClosed', 'DealOrderAlreadyLocked', 'DealOrderMustBeLocked', 'DuplicateDealOrder', 'DealOrderExpired', 'AskOrderExpired', 'BidOrderExpired', 'OfferExpired', 'AskBidMismatch', 'SameOwner', 'InvalidSignature', 'NotBorrower', 'MalformedDealOrder', 'NotLender', 'ScaleDecodeError', 'UnverifiedTransferPoolFull', 'RepaymentOrderUnsupported', 'NotLegacyWalletOwner', 'LegacySighashMalformed', 'LegacyWalletNotFound', 'LegacyBalanceKeeperMissing', 'VerifyStringTooLong', 'GuidAlreadyUsed', 'InvalidMaturity']
+    _enum: ['AddressAlreadyRegistered', 'NonExistentAddress', 'NonExistentDealOrder', 'NonExistentAskOrder', 'NonExistentBidOrder', 'NonExistentOffer', 'NonExistentTransfer', 'TransferAlreadyRegistered', 'TransferMismatch', 'TransferAlreadyProcessed', 'TransferAmountInsufficient', 'MalformedTransfer', 'UnsupportedTransferKind', 'InsufficientAuthority', 'DuplicateId', 'NotAddressOwner', 'OffchainSignedTxFailed', 'NoLocalAcctForSignedTx', 'RepaymentOrderNonZeroGain', 'AddressPlatformMismatch', 'AlreadyAuthority', 'DuplicateOffer', 'DealNotFunded', 'DealOrderAlreadyFunded', 'DealOrderAlreadyClosed', 'DealOrderAlreadyLocked', 'DealOrderMustBeLocked', 'DuplicateDealOrder', 'DealOrderExpired', 'AskOrderExpired', 'BidOrderExpired', 'OfferExpired', 'AskBidMismatch', 'SameOwner', 'InvalidSignature', 'NotBorrower', 'MalformedDealOrder', 'NotLender', 'UnverifiedTransferPoolFull', 'RepaymentOrderUnsupported', 'NotLegacyWalletOwner', 'LegacyWalletNotFound', 'LegacyBalanceKeeperMissing', 'GuidAlreadyUsed', 'InvalidMaturity']
   },
   /**
    * Lookup128: pallet_difficulty::DifficultyAndTimestamp<Moment>
diff --git a/java-script/src/interfaces/types-lookup.ts b/java-script/src/interfaces/types-lookup.ts
index d06db68..d07ef35 100644
--- a/java-script/src/interfaces/types-lookup.ts
+++ b/java-script/src/interfaces/types-lookup.ts
@@ -806,7 +806,6 @@ declare module '@polkadot/types/lookup' {
     readonly isMalformedTransfer: boolean;
     readonly isUnsupportedTransferKind: boolean;
     readonly isInsufficientAuthority: boolean;
-    readonly isNonExistentRepaymentOrder: boolean;
     readonly isDuplicateId: boolean;
     readonly isNotAddressOwner: boolean;
     readonly isOffchainSignedTxFailed: boolean;
@@ -831,17 +830,14 @@ declare module '@polkadot/types/lookup' {
     readonly isNotBorrower: boolean;
     readonly isMalformedDealOrder: boolean;
     readonly isNotLender: boolean;
-    readonly isScaleDecodeError: boolean;
     readonly isUnverifiedTransferPoolFull: boolean;
     readonly isRepaymentOrderUnsupported: boolean;
     readonly isNotLegacyWalletOwner: boolean;
-    readonly isLegacySighashMalformed: boolean;
     readonly isLegacyWalletNotFound: boolean;
     readonly isLegacyBalanceKeeperMissing: boolean;
-    readonly isVerifyStringTooLong: boolean;
     readonly isGuidAlreadyUsed: boolean;
     readonly isInvalidMaturity: boolean;
-    readonly type: 'AddressAlreadyRegistered' | 'NonExistentAddress' | 'NonExistentDealOrder' | 'NonExistentAskOrder' | 'NonExistentBidOrder' | 'NonExistentOffer' | 'NonExistentTransfer' | 'TransferAlreadyRegistered' | 'TransferMismatch' | 'TransferAlreadyProcessed' | 'TransferAmountInsufficient' | 'MalformedTransfer' | 'UnsupportedTransferKind' | 'InsufficientAuthority' | 'NonExistentRepaymentOrder' | 'DuplicateId' | 'NotAddressOwner' | 'OffchainSignedTxFailed' | 'NoLocalAcctForSignedTx' | 'RepaymentOrderNonZeroGain' | 'AddressPlatformMismatch' | 'AlreadyAuthority' | 'DuplicateOffer' | 'DealNotFunded' | 'DealOrderAlreadyFunded' | 'DealOrderAlreadyClosed' | 'DealOrderAlreadyLocked' | 'DealOrderMustBeLocked' | 'DuplicateDealOrder' | 'DealOrderExpired' | 'AskOrderExpired' | 'BidOrderExpired' | 'OfferExpired' | 'AskBidMismatch' | 'SameOwner' | 'InvalidSignature' | 'NotBorrower' | 'MalformedDealOrder' | 'NotLender' | 'ScaleDecodeError' | 'UnverifiedTransferPoolFull' | 'RepaymentOrderUnsupported' | 'NotLegacyWalletOwner' | 'LegacySighashMalformed' | 'LegacyWalletNotFound' | 'LegacyBalanceKeeperMissing' | 'VerifyStringTooLong' | 'GuidAlreadyUsed' | 'InvalidMaturity';
+    readonly type: 'AddressAlreadyRegistered' | 'NonExistentAddress' | 'NonExistentDealOrder' | 'NonExistentAskOrder' | 'NonExistentBidOrder' | 'NonExistentOffer' | 'NonExistentTransfer' | 'TransferAlreadyRegistered' | 'TransferMismatch' | 'TransferAlreadyProcessed' | 'TransferAmountInsufficient' | 'MalformedTransfer' | 'UnsupportedTransferKind' | 'InsufficientAuthority' | 'DuplicateId' | 'NotAddressOwner' | 'OffchainSignedTxFailed' | 'NoLocalAcctForSignedTx' | 'RepaymentOrderNonZeroGain' | 'AddressPlatformMismatch' | 'AlreadyAuthority' | 'DuplicateOffer' | 'DealNotFunded' | 'DealOrderAlreadyFunded' | 'DealOrderAlreadyClosed' | 'DealOrderAlreadyLocked' | 'DealOrderMustBeLocked' | 'DuplicateDealOrder' | 'DealOrderExpired' | 'AskOrderExpired' | 'BidOrderExpired' | 'OfferExpired' | 'AskBidMismatch' | 'SameOwner' | 'InvalidSignature' | 'NotBorrower' | 'MalformedDealOrder' | 'NotLender' | 'UnverifiedTransferPoolFull' | 'RepaymentOrderUnsupported' | 'NotLegacyWalletOwner' | 'LegacyWalletNotFound' | 'LegacyBalanceKeeperMissing' | 'GuidAlreadyUsed' | 'InvalidMaturity';
   }
 
   /** @name PalletDifficultyDifficultyAndTimestamp (128) */
```

@jfrank-summit, @nathanwhit what do you want to to with this diff? Should I commit it in git (and also add a check in CI for that) or ignore it ? 

Tests and examples are able to execute successfully either way.

Reply:
FTR we agree that we should not commit this in git. For examples (e.g. immediate consumption from a git checkout) the typedefs are coming from testnet.